[INFO][2018-09-1123:02:34][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [spring/spring-mybatis.xml]
[INFO][2018-09-1123:02:34][org.springframework.context.support.GenericApplicationContext]Refreshing org.springframework.context.support.GenericApplicationContext@7921b0a2: startup date [Tue Sep 11 23:02:34 CST 2018]; root of context hierarchy
[INFO][2018-09-1123:02:34][org.springframework.beans.factory.config.PropertyPlaceholderConfigurer]Loading properties file from class path resource [properties/jdbc.properties]
[INFO][2018-09-1123:02:34][org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor]JSR-330 'javax.inject.Inject' annotation found and supported for autowiring
[INFO][2018-09-1123:02:35][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [org/springframework/jdbc/support/sql-error-codes.xml]
[INFO][2018-09-1123:02:35][org.springframework.jdbc.support.SQLErrorCodesFactory]SQLErrorCodes loaded: [DB2, Derby, H2, HSQL, Informix, MS-SQL, MySQL, Oracle, PostgreSQL, Sybase]
[INFO][2018-09-1123:02:35][org.springframework.context.support.GenericApplicationContext]Closing org.springframework.context.support.GenericApplicationContext@7921b0a2: startup date [Tue Sep 11 23:02:34 CST 2018]; root of context hierarchy
[INFO][2018-09-1123:06:50][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [spring/spring-mybatis.xml]
[INFO][2018-09-1123:06:50][org.springframework.context.support.GenericApplicationContext]Refreshing org.springframework.context.support.GenericApplicationContext@7921b0a2: startup date [Tue Sep 11 23:06:50 CST 2018]; root of context hierarchy
[INFO][2018-09-1123:06:51][org.springframework.beans.factory.config.PropertyPlaceholderConfigurer]Loading properties file from class path resource [properties/jdbc.properties]
[INFO][2018-09-1123:06:51][org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor]JSR-330 'javax.inject.Inject' annotation found and supported for autowiring
[INFO][2018-09-1123:06:52][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [org/springframework/jdbc/support/sql-error-codes.xml]
[INFO][2018-09-1123:06:52][org.springframework.jdbc.support.SQLErrorCodesFactory]SQLErrorCodes loaded: [DB2, Derby, H2, HSQL, Informix, MS-SQL, MySQL, Oracle, PostgreSQL, Sybase]
[INFO][2018-09-1123:06:52][org.springframework.context.support.GenericApplicationContext]Closing org.springframework.context.support.GenericApplicationContext@7921b0a2: startup date [Tue Sep 11 23:06:50 CST 2018]; root of context hierarchy
[INFO][2018-09-1123:24:09][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [spring/spring-mybatis.xml]
[INFO][2018-09-1123:24:10][org.springframework.context.support.GenericApplicationContext]Refreshing org.springframework.context.support.GenericApplicationContext@573f2bb1: startup date [Tue Sep 11 23:24:10 CST 2018]; root of context hierarchy
[INFO][2018-09-1123:24:10][org.springframework.beans.factory.config.PropertyPlaceholderConfigurer]Loading properties file from class path resource [properties/jdbc.properties]
[INFO][2018-09-1123:24:10][org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor]JSR-330 'javax.inject.Inject' annotation found and supported for autowiring
[INFO][2018-09-1123:24:11][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Began transaction (1) for test context [DefaultTestContext@5ab956d7 testClass = UserTest, testInstance = mapper.UserTest@3646a422, testMethod = testInsertUser@UserTest, testException = [null], mergedContextConfiguration = [MergedContextConfiguration@750e2b97 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]; transaction manager [org.springframework.jdbc.datasource.DataSourceTransactionManager@3e27aa33]; rollback [true]
[INFO][2018-09-1123:24:11][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [org/springframework/jdbc/support/sql-error-codes.xml]
[INFO][2018-09-1123:24:11][org.springframework.jdbc.support.SQLErrorCodesFactory]SQLErrorCodes loaded: [DB2, Derby, H2, HSQL, Informix, MS-SQL, MySQL, Oracle, PostgreSQL, Sybase]
[INFO][2018-09-1123:24:11][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Rolled back transaction after test execution for test context [DefaultTestContext@5ab956d7 testClass = UserTest, testInstance = mapper.UserTest@3646a422, testMethod = testInsertUser@UserTest, testException = org.springframework.dao.DataIntegrityViolationException: 
### Error updating database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Column 'user_password_hash' cannot be null
### The error may involve com.mayousheng.www.mapper.UserMapper.insert-Inline
### The error occurred while setting parameters
### SQL: insert into user (user_id, user_phone, user_password_hash,        user_agreement, user_regist_time, user_last_login_time,        subscribe_count, follower_count, receive_like_count,        receive_favorite_count, receive_worth_count,        receive_unworth_count, receive_upmarket_count,        receive_downmarket_count, receive_tip_count,        receive_comment_count, give_like_count, give_favorite_count,        give_worth_count, give_unworth_count, give_upmarket_count,        give_downmarket_count, give_tip_count, give_comment_count,        content_count, trend_count, credit_score,        nick_name, level, experience,        coin, avatar, token,        token_reserved_a, token_reserved_b, sex,        birthday, location, position,        about_me, married, sexual_orientation,        address, occupation, school,        hometown, mail, page_view,        banned, phone_verified, mail_verified       )     values (?, ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?,        ?, ?,        ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?, ?       )
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Column 'user_password_hash' cannot be null
; SQL []; Column 'user_password_hash' cannot be null; nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Column 'user_password_hash' cannot be null, mergedContextConfiguration = [MergedContextConfiguration@750e2b97 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]
[INFO][2018-09-1123:24:11][org.springframework.context.support.GenericApplicationContext]Closing org.springframework.context.support.GenericApplicationContext@573f2bb1: startup date [Tue Sep 11 23:24:10 CST 2018]; root of context hierarchy
[INFO][2018-09-1219:38:10][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [spring/spring-mybatis.xml]
[INFO][2018-09-1219:38:11][org.springframework.context.support.GenericApplicationContext]Refreshing org.springframework.context.support.GenericApplicationContext@573f2bb1: startup date [Wed Sep 12 19:38:11 CST 2018]; root of context hierarchy
[INFO][2018-09-1219:38:11][org.springframework.beans.factory.config.PropertyPlaceholderConfigurer]Loading properties file from class path resource [properties/jdbc.properties]
[INFO][2018-09-1219:38:11][org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor]JSR-330 'javax.inject.Inject' annotation found and supported for autowiring
[INFO][2018-09-1219:38:12][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Began transaction (1) for test context [DefaultTestContext@5ab956d7 testClass = UserTest, testInstance = mapper.UserTest@3646a422, testMethod = testInsertUser@UserTest, testException = [null], mergedContextConfiguration = [MergedContextConfiguration@750e2b97 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]; transaction manager [org.springframework.jdbc.datasource.DataSourceTransactionManager@3e27aa33]; rollback [true]
[INFO][2018-09-1219:38:12][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [org/springframework/jdbc/support/sql-error-codes.xml]
[INFO][2018-09-1219:38:12][org.springframework.jdbc.support.SQLErrorCodesFactory]SQLErrorCodes loaded: [DB2, Derby, H2, HSQL, Informix, MS-SQL, MySQL, Oracle, PostgreSQL, Sybase]
[INFO][2018-09-1219:38:12][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Rolled back transaction after test execution for test context [DefaultTestContext@5ab956d7 testClass = UserTest, testInstance = mapper.UserTest@3646a422, testMethod = testInsertUser@UserTest, testException = org.springframework.dao.DataIntegrityViolationException: 
### Error updating database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Column 'user_password_hash' cannot be null
### The error may involve com.mayousheng.www.mapper.UserMapper.insert-Inline
### The error occurred while setting parameters
### SQL: insert into user (user_id, user_phone, user_password_hash,        user_agreement, user_regist_time, user_last_login_time,        subscribe_count, follower_count, receive_like_count,        receive_favorite_count, receive_worth_count,        receive_unworth_count, receive_upmarket_count,        receive_downmarket_count, receive_tip_count,        receive_comment_count, give_like_count, give_favorite_count,        give_worth_count, give_unworth_count, give_upmarket_count,        give_downmarket_count, give_tip_count, give_comment_count,        content_count, trend_count, credit_score,        nick_name, level, experience,        coin, avatar, token,        token_reserved_a, token_reserved_b, sex,        birthday, location, position,        about_me, married, sexual_orientation,        address, occupation, school,        hometown, mail, page_view,        banned, phone_verified, mail_verified       )     values (?, ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?,        ?, ?,        ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?, ?       )
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Column 'user_password_hash' cannot be null
; SQL []; Column 'user_password_hash' cannot be null; nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Column 'user_password_hash' cannot be null, mergedContextConfiguration = [MergedContextConfiguration@750e2b97 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]
[INFO][2018-09-1219:38:12][org.springframework.context.support.GenericApplicationContext]Closing org.springframework.context.support.GenericApplicationContext@573f2bb1: startup date [Wed Sep 12 19:38:11 CST 2018]; root of context hierarchy
[INFO][2018-09-1219:38:39][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [spring/spring-mybatis.xml]
[INFO][2018-09-1219:38:39][org.springframework.context.support.GenericApplicationContext]Refreshing org.springframework.context.support.GenericApplicationContext@573f2bb1: startup date [Wed Sep 12 19:38:39 CST 2018]; root of context hierarchy
[INFO][2018-09-1219:38:40][org.springframework.beans.factory.config.PropertyPlaceholderConfigurer]Loading properties file from class path resource [properties/jdbc.properties]
[INFO][2018-09-1219:38:40][org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor]JSR-330 'javax.inject.Inject' annotation found and supported for autowiring
[INFO][2018-09-1219:38:41][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Began transaction (1) for test context [DefaultTestContext@5ab956d7 testClass = UserTest, testInstance = mapper.UserTest@3646a422, testMethod = testInsertUser@UserTest, testException = [null], mergedContextConfiguration = [MergedContextConfiguration@750e2b97 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]; transaction manager [org.springframework.jdbc.datasource.DataSourceTransactionManager@3e27aa33]; rollback [true]
[INFO][2018-09-1219:38:41][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [org/springframework/jdbc/support/sql-error-codes.xml]
[INFO][2018-09-1219:38:41][org.springframework.jdbc.support.SQLErrorCodesFactory]SQLErrorCodes loaded: [DB2, Derby, H2, HSQL, Informix, MS-SQL, MySQL, Oracle, PostgreSQL, Sybase]
[INFO][2018-09-1219:38:41][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Rolled back transaction after test execution for test context [DefaultTestContext@5ab956d7 testClass = UserTest, testInstance = mapper.UserTest@3646a422, testMethod = testInsertUser@UserTest, testException = org.springframework.dao.DataIntegrityViolationException: 
### Error updating database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Column 'user_password_hash' cannot be null
### The error may involve com.mayousheng.www.mapper.UserMapper.insert-Inline
### The error occurred while setting parameters
### SQL: insert into user (user_id, user_phone, user_password_hash,        user_agreement, user_regist_time, user_last_login_time,        subscribe_count, follower_count, receive_like_count,        receive_favorite_count, receive_worth_count,        receive_unworth_count, receive_upmarket_count,        receive_downmarket_count, receive_tip_count,        receive_comment_count, give_like_count, give_favorite_count,        give_worth_count, give_unworth_count, give_upmarket_count,        give_downmarket_count, give_tip_count, give_comment_count,        content_count, trend_count, credit_score,        nick_name, level, experience,        coin, avatar, token,        token_reserved_a, token_reserved_b, sex,        birthday, location, position,        about_me, married, sexual_orientation,        address, occupation, school,        hometown, mail, page_view,        banned, phone_verified, mail_verified       )     values (?, ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?,        ?, ?,        ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?, ?       )
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Column 'user_password_hash' cannot be null
; SQL []; Column 'user_password_hash' cannot be null; nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Column 'user_password_hash' cannot be null, mergedContextConfiguration = [MergedContextConfiguration@750e2b97 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]
[INFO][2018-09-1219:38:41][org.springframework.context.support.GenericApplicationContext]Closing org.springframework.context.support.GenericApplicationContext@573f2bb1: startup date [Wed Sep 12 19:38:39 CST 2018]; root of context hierarchy
[INFO][2018-09-1219:40:29][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [spring/spring-mybatis.xml]
[INFO][2018-09-1219:40:29][org.springframework.context.support.GenericApplicationContext]Refreshing org.springframework.context.support.GenericApplicationContext@5c1a8622: startup date [Wed Sep 12 19:40:29 CST 2018]; root of context hierarchy
[INFO][2018-09-1219:40:29][org.springframework.beans.factory.config.PropertyPlaceholderConfigurer]Loading properties file from class path resource [properties/jdbc.properties]
[INFO][2018-09-1219:40:29][org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor]JSR-330 'javax.inject.Inject' annotation found and supported for autowiring
[INFO][2018-09-1219:40:30][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Began transaction (1) for test context [DefaultTestContext@5b6813df testClass = UserTest, testInstance = mapper.UserTest@5f2606b, testMethod = testInsertUser@UserTest, testException = [null], mergedContextConfiguration = [MergedContextConfiguration@2b58f754 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]; transaction manager [org.springframework.jdbc.datasource.DataSourceTransactionManager@3ebff828]; rollback [true]
[INFO][2018-09-1219:40:30][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [org/springframework/jdbc/support/sql-error-codes.xml]
[INFO][2018-09-1219:40:30][org.springframework.jdbc.support.SQLErrorCodesFactory]SQLErrorCodes loaded: [DB2, Derby, H2, HSQL, Informix, MS-SQL, MySQL, Oracle, PostgreSQL, Sybase]
[INFO][2018-09-1219:40:30][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Rolled back transaction after test execution for test context [DefaultTestContext@5b6813df testClass = UserTest, testInstance = mapper.UserTest@5f2606b, testMethod = testInsertUser@UserTest, testException = org.springframework.dao.DataIntegrityViolationException: 
### Error updating database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Column 'user_password_hash' cannot be null
### The error may involve com.mayousheng.www.mapper.UserMapper.insert-Inline
### The error occurred while setting parameters
### SQL: insert into user (user_id, user_phone, user_password_hash,        user_agreement, user_regist_time, user_last_login_time,        subscribe_count, follower_count, receive_like_count,        receive_favorite_count, receive_worth_count,        receive_unworth_count, receive_upmarket_count,        receive_downmarket_count, receive_tip_count,        receive_comment_count, give_like_count, give_favorite_count,        give_worth_count, give_unworth_count, give_upmarket_count,        give_downmarket_count, give_tip_count, give_comment_count,        content_count, trend_count, credit_score,        nick_name, level, experience,        coin, avatar, token,        token_reserved_a, token_reserved_b, sex,        birthday, location, position,        about_me, married, sexual_orientation,        address, occupation, school,        hometown, mail, page_view,        banned, phone_verified, mail_verified       )     values (?, ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?,        ?, ?,        ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?, ?       )
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Column 'user_password_hash' cannot be null
; SQL []; Column 'user_password_hash' cannot be null; nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Column 'user_password_hash' cannot be null, mergedContextConfiguration = [MergedContextConfiguration@2b58f754 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]
[INFO][2018-09-1219:40:30][org.springframework.context.support.GenericApplicationContext]Closing org.springframework.context.support.GenericApplicationContext@5c1a8622: startup date [Wed Sep 12 19:40:29 CST 2018]; root of context hierarchy
[INFO][2018-09-1220:01:22][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [spring/spring-mybatis.xml]
[INFO][2018-09-1220:01:22][org.springframework.context.support.GenericApplicationContext]Refreshing org.springframework.context.support.GenericApplicationContext@573f2bb1: startup date [Wed Sep 12 20:01:22 CST 2018]; root of context hierarchy
[INFO][2018-09-1220:01:22][org.springframework.beans.factory.config.PropertyPlaceholderConfigurer]Loading properties file from class path resource [properties/jdbc.properties]
[INFO][2018-09-1220:01:22][org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor]JSR-330 'javax.inject.Inject' annotation found and supported for autowiring
[INFO][2018-09-1220:01:23][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Began transaction (1) for test context [DefaultTestContext@5ab956d7 testClass = UserTest, testInstance = mapper.UserTest@3646a422, testMethod = testSelectUser@UserTest, testException = [null], mergedContextConfiguration = [MergedContextConfiguration@750e2b97 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]; transaction manager [org.springframework.jdbc.datasource.DataSourceTransactionManager@3e27aa33]; rollback [true]
[INFO][2018-09-1220:01:23][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Rolled back transaction after test execution for test context [DefaultTestContext@5ab956d7 testClass = UserTest, testInstance = mapper.UserTest@3646a422, testMethod = testSelectUser@UserTest, testException = [null], mergedContextConfiguration = [MergedContextConfiguration@750e2b97 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]
[INFO][2018-09-1220:01:23][org.springframework.context.support.GenericApplicationContext]Closing org.springframework.context.support.GenericApplicationContext@573f2bb1: startup date [Wed Sep 12 20:01:22 CST 2018]; root of context hierarchy
[INFO][2018-09-1220:02:01][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [spring/spring-mybatis.xml]
[INFO][2018-09-1220:02:01][org.springframework.context.support.GenericApplicationContext]Refreshing org.springframework.context.support.GenericApplicationContext@573f2bb1: startup date [Wed Sep 12 20:02:01 CST 2018]; root of context hierarchy
[INFO][2018-09-1220:02:01][org.springframework.beans.factory.config.PropertyPlaceholderConfigurer]Loading properties file from class path resource [properties/jdbc.properties]
[INFO][2018-09-1220:02:01][org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor]JSR-330 'javax.inject.Inject' annotation found and supported for autowiring
[INFO][2018-09-1220:02:02][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Began transaction (1) for test context [DefaultTestContext@5ab956d7 testClass = UserTest, testInstance = mapper.UserTest@3646a422, testMethod = testSelectUser@UserTest, testException = [null], mergedContextConfiguration = [MergedContextConfiguration@750e2b97 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]; transaction manager [org.springframework.jdbc.datasource.DataSourceTransactionManager@3e27aa33]; rollback [true]
[INFO][2018-09-1220:02:02][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Rolled back transaction after test execution for test context [DefaultTestContext@5ab956d7 testClass = UserTest, testInstance = mapper.UserTest@3646a422, testMethod = testSelectUser@UserTest, testException = [null], mergedContextConfiguration = [MergedContextConfiguration@750e2b97 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]
[INFO][2018-09-1220:02:02][org.springframework.context.support.GenericApplicationContext]Closing org.springframework.context.support.GenericApplicationContext@573f2bb1: startup date [Wed Sep 12 20:02:01 CST 2018]; root of context hierarchy
[INFO][2018-09-1220:05:31][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [spring/spring-mybatis.xml]
[INFO][2018-09-1220:05:32][org.springframework.context.support.GenericApplicationContext]Refreshing org.springframework.context.support.GenericApplicationContext@573f2bb1: startup date [Wed Sep 12 20:05:32 CST 2018]; root of context hierarchy
[INFO][2018-09-1220:05:32][org.springframework.beans.factory.config.PropertyPlaceholderConfigurer]Loading properties file from class path resource [properties/jdbc.properties]
[INFO][2018-09-1220:05:32][org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor]JSR-330 'javax.inject.Inject' annotation found and supported for autowiring
[INFO][2018-09-1220:05:33][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Began transaction (1) for test context [DefaultTestContext@5ab956d7 testClass = UserTest, testInstance = mapper.UserTest@3646a422, testMethod = testInsertUser@UserTest, testException = [null], mergedContextConfiguration = [MergedContextConfiguration@750e2b97 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]; transaction manager [org.springframework.jdbc.datasource.DataSourceTransactionManager@3e27aa33]; rollback [true]
[INFO][2018-09-1220:05:33][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [org/springframework/jdbc/support/sql-error-codes.xml]
[INFO][2018-09-1220:05:33][org.springframework.jdbc.support.SQLErrorCodesFactory]SQLErrorCodes loaded: [DB2, Derby, H2, HSQL, Informix, MS-SQL, MySQL, Oracle, PostgreSQL, Sybase]
[INFO][2018-09-1220:05:33][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Rolled back transaction after test execution for test context [DefaultTestContext@5ab956d7 testClass = UserTest, testInstance = mapper.UserTest@3646a422, testMethod = testInsertUser@UserTest, testException = org.springframework.dao.DataIntegrityViolationException: 
### Error updating database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Column 'user_agreement' cannot be null
### The error may involve com.mayousheng.www.mapper.UserMapper.insert-Inline
### The error occurred while setting parameters
### SQL: insert into user (user_id, user_phone, user_password_hash,        user_agreement, user_regist_time, user_last_login_time,        subscribe_count, follower_count, receive_like_count,        receive_favorite_count, receive_worth_count,        receive_unworth_count, receive_upmarket_count,        receive_downmarket_count, receive_tip_count,        receive_comment_count, give_like_count, give_favorite_count,        give_worth_count, give_unworth_count, give_upmarket_count,        give_downmarket_count, give_tip_count, give_comment_count,        content_count, trend_count, credit_score,        nick_name, level, experience,        coin, avatar, token,        token_reserved_a, token_reserved_b, sex,        birthday, location, position,        about_me, married, sexual_orientation,        address, occupation, school,        hometown, mail, page_view,        banned, phone_verified, mail_verified       )     values (?, ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?,        ?, ?,        ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?, ?,        ?, ?, ?       )
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Column 'user_agreement' cannot be null
; SQL []; Column 'user_agreement' cannot be null; nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Column 'user_agreement' cannot be null, mergedContextConfiguration = [MergedContextConfiguration@750e2b97 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]
[INFO][2018-09-1220:05:33][org.springframework.context.support.GenericApplicationContext]Closing org.springframework.context.support.GenericApplicationContext@573f2bb1: startup date [Wed Sep 12 20:05:32 CST 2018]; root of context hierarchy
[INFO][2018-09-1220:11:05][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [spring/spring-mybatis.xml]
[INFO][2018-09-1220:11:05][org.springframework.context.support.GenericApplicationContext]Refreshing org.springframework.context.support.GenericApplicationContext@573f2bb1: startup date [Wed Sep 12 20:11:05 CST 2018]; root of context hierarchy
[INFO][2018-09-1220:11:05][org.springframework.beans.factory.config.PropertyPlaceholderConfigurer]Loading properties file from class path resource [properties/jdbc.properties]
[INFO][2018-09-1220:11:05][org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor]JSR-330 'javax.inject.Inject' annotation found and supported for autowiring
[INFO][2018-09-1220:11:06][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Began transaction (1) for test context [DefaultTestContext@5ab956d7 testClass = UserTest, testInstance = mapper.UserTest@3646a422, testMethod = testInsertUser@UserTest, testException = [null], mergedContextConfiguration = [MergedContextConfiguration@750e2b97 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]; transaction manager [org.springframework.jdbc.datasource.DataSourceTransactionManager@3e27aa33]; rollback [true]
[INFO][2018-09-1220:11:06][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Rolled back transaction after test execution for test context [DefaultTestContext@5ab956d7 testClass = UserTest, testInstance = mapper.UserTest@3646a422, testMethod = testInsertUser@UserTest, testException = [null], mergedContextConfiguration = [MergedContextConfiguration@750e2b97 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]
[INFO][2018-09-1220:11:06][org.springframework.context.support.GenericApplicationContext]Closing org.springframework.context.support.GenericApplicationContext@573f2bb1: startup date [Wed Sep 12 20:11:05 CST 2018]; root of context hierarchy
[INFO][2018-09-1220:12:01][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [spring/spring-mybatis.xml]
[INFO][2018-09-1220:12:01][org.springframework.context.support.GenericApplicationContext]Refreshing org.springframework.context.support.GenericApplicationContext@573f2bb1: startup date [Wed Sep 12 20:12:01 CST 2018]; root of context hierarchy
[INFO][2018-09-1220:12:01][org.springframework.beans.factory.config.PropertyPlaceholderConfigurer]Loading properties file from class path resource [properties/jdbc.properties]
[INFO][2018-09-1220:12:01][org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor]JSR-330 'javax.inject.Inject' annotation found and supported for autowiring
[INFO][2018-09-1220:12:02][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Began transaction (1) for test context [DefaultTestContext@5ab956d7 testClass = UserTest, testInstance = mapper.UserTest@3646a422, testMethod = testInsertUser@UserTest, testException = [null], mergedContextConfiguration = [MergedContextConfiguration@750e2b97 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]; transaction manager [org.springframework.jdbc.datasource.DataSourceTransactionManager@3e27aa33]; rollback [true]
[INFO][2018-09-1220:12:02][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [org/springframework/jdbc/support/sql-error-codes.xml]
[INFO][2018-09-1220:12:02][org.springframework.jdbc.support.SQLErrorCodesFactory]SQLErrorCodes loaded: [DB2, Derby, H2, HSQL, Informix, MS-SQL, MySQL, Oracle, PostgreSQL, Sybase]
[INFO][2018-09-1220:12:02][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Rolled back transaction after test execution for test context [DefaultTestContext@5ab956d7 testClass = UserTest, testInstance = mapper.UserTest@3646a422, testMethod = testInsertUser@UserTest, testException = org.springframework.dao.DuplicateKeyException: 
### Error updating database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '17816875157' for key 'phone_unique'
### The error may involve com.mayousheng.www.mapper.UserMapper.insertSelective-Inline
### The error occurred while setting parameters
### SQL: insert into user      ( user_phone,                       user_password_hash,                              user_regist_time,                       user_last_login_time,                                                                                                                                                                          nick_name,                                            avatar,                       token )       values ( ?,                       ?,                              ?,                       ?,                                                                                                                                                                          ?,                                            ?,                       ? )
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '17816875157' for key 'phone_unique'
; SQL []; Duplicate entry '17816875157' for key 'phone_unique'; nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '17816875157' for key 'phone_unique', mergedContextConfiguration = [MergedContextConfiguration@750e2b97 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]
[INFO][2018-09-1220:12:02][org.springframework.context.support.GenericApplicationContext]Closing org.springframework.context.support.GenericApplicationContext@573f2bb1: startup date [Wed Sep 12 20:12:01 CST 2018]; root of context hierarchy
[INFO][2018-09-1222:52:15][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [spring/spring-mybatis.xml]
[INFO][2018-09-1222:52:15][org.springframework.context.support.GenericApplicationContext]Refreshing org.springframework.context.support.GenericApplicationContext@6d8a00e3: startup date [Wed Sep 12 22:52:15 CST 2018]; root of context hierarchy
[INFO][2018-09-1222:52:16][org.springframework.beans.factory.config.PropertyPlaceholderConfigurer]Loading properties file from class path resource [properties/jdbc.properties]
[INFO][2018-09-1222:52:16][org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor]JSR-330 'javax.inject.Inject' annotation found and supported for autowiring
[INFO][2018-09-1222:52:16][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Began transaction (1) for test context [DefaultTestContext@76a4ebf2 testClass = UserTest, testInstance = mapper.UserTest@2e8c1c9b, testMethod = testInsertUser@UserTest, testException = [null], mergedContextConfiguration = [MergedContextConfiguration@53fe15ff testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]; transaction manager [org.springframework.jdbc.datasource.DataSourceTransactionManager@449a4f23]; rollback [true]
[INFO][2018-09-1222:52:17][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [org/springframework/jdbc/support/sql-error-codes.xml]
[INFO][2018-09-1222:52:17][org.springframework.jdbc.support.SQLErrorCodesFactory]SQLErrorCodes loaded: [DB2, Derby, H2, HSQL, Informix, MS-SQL, MySQL, Oracle, PostgreSQL, Sybase]
[INFO][2018-09-1222:52:17][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Rolled back transaction after test execution for test context [DefaultTestContext@76a4ebf2 testClass = UserTest, testInstance = mapper.UserTest@2e8c1c9b, testMethod = testInsertUser@UserTest, testException = org.springframework.dao.DuplicateKeyException: 
### Error updating database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '17816875157' for key 'phone_unique'
### The error may involve com.mayousheng.www.mapper.UserMapper.insertSelective-Inline
### The error occurred while setting parameters
### SQL: insert into user      ( user_phone,                       user_password_hash,                              user_regist_time,                       user_last_login_time,                                                                                                                                                                          nick_name,                                            avatar,                       token )       values ( ?,                       ?,                              ?,                       ?,                                                                                                                                                                          ?,                                            ?,                       ? )
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '17816875157' for key 'phone_unique'
; SQL []; Duplicate entry '17816875157' for key 'phone_unique'; nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '17816875157' for key 'phone_unique', mergedContextConfiguration = [MergedContextConfiguration@53fe15ff testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]
[INFO][2018-09-1222:52:17][org.springframework.context.support.GenericApplicationContext]Closing org.springframework.context.support.GenericApplicationContext@6d8a00e3: startup date [Wed Sep 12 22:52:15 CST 2018]; root of context hierarchy
[INFO][2018-10-3121:33:16][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [spring/spring-mybatis.xml]
[INFO][2018-10-3121:33:17][org.springframework.context.support.GenericApplicationContext]Refreshing org.springframework.context.support.GenericApplicationContext@15761df8: startup date [Wed Oct 31 21:33:17 CST 2018]; root of context hierarchy
[INFO][2018-10-3121:33:17][org.springframework.beans.factory.config.PropertyPlaceholderConfigurer]Loading properties file from class path resource [properties/jdbc.properties]
[INFO][2018-10-3121:33:17][org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor]JSR-330 'javax.inject.Inject' annotation found and supported for autowiring
[INFO][2018-10-3121:33:18][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Began transaction (1) for test context [DefaultTestContext@3a1d593e testClass = UserTest, testInstance = mapper.UserTest@4a8a60bc, testMethod = testdeletenotfound@UserTest, testException = [null], mergedContextConfiguration = [MergedContextConfiguration@361c294e testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]; transaction manager [org.springframework.jdbc.datasource.DataSourceTransactionManager@7859e786]; rollback [true]
[INFO][2018-10-3121:33:18][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Rolled back transaction after test execution for test context [DefaultTestContext@3a1d593e testClass = UserTest, testInstance = mapper.UserTest@4a8a60bc, testMethod = testdeletenotfound@UserTest, testException = [null], mergedContextConfiguration = [MergedContextConfiguration@361c294e testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]
[INFO][2018-10-3121:33:18][org.springframework.context.support.GenericApplicationContext]Closing org.springframework.context.support.GenericApplicationContext@15761df8: startup date [Wed Oct 31 21:33:17 CST 2018]; root of context hierarchy
[INFO][2018-10-3121:37:13][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [spring/spring-mybatis.xml]
[INFO][2018-10-3121:37:14][org.springframework.context.support.GenericApplicationContext]Refreshing org.springframework.context.support.GenericApplicationContext@15761df8: startup date [Wed Oct 31 21:37:14 CST 2018]; root of context hierarchy
[INFO][2018-10-3121:37:14][org.springframework.beans.factory.config.PropertyPlaceholderConfigurer]Loading properties file from class path resource [properties/jdbc.properties]
[INFO][2018-10-3121:37:14][org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor]JSR-330 'javax.inject.Inject' annotation found and supported for autowiring
[INFO][2018-10-3121:37:15][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Began transaction (1) for test context [DefaultTestContext@3a1d593e testClass = UserTest, testInstance = mapper.UserTest@4a8a60bc, testMethod = testdeletenotfound@UserTest, testException = [null], mergedContextConfiguration = [MergedContextConfiguration@361c294e testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]; transaction manager [org.springframework.jdbc.datasource.DataSourceTransactionManager@7859e786]; rollback [true]
[INFO][2018-10-3121:37:15][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Rolled back transaction after test execution for test context [DefaultTestContext@3a1d593e testClass = UserTest, testInstance = mapper.UserTest@4a8a60bc, testMethod = testdeletenotfound@UserTest, testException = [null], mergedContextConfiguration = [MergedContextConfiguration@361c294e testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]
[INFO][2018-10-3121:37:15][org.springframework.context.support.GenericApplicationContext]Closing org.springframework.context.support.GenericApplicationContext@15761df8: startup date [Wed Oct 31 21:37:14 CST 2018]; root of context hierarchy
[INFO][2018-11-0114:07:41][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [spring/spring-mybatis.xml]
[INFO][2018-11-0114:07:41][org.springframework.context.support.GenericApplicationContext]Refreshing org.springframework.context.support.GenericApplicationContext@5dfcfece: startup date [Thu Nov 01 14:07:41 CST 2018]; root of context hierarchy
[INFO][2018-11-0114:07:42][org.springframework.beans.factory.config.PropertyPlaceholderConfigurer]Loading properties file from class path resource [properties/jdbc.properties]
[INFO][2018-11-0114:07:42][org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor]JSR-330 'javax.inject.Inject' annotation found and supported for autowiring
[INFO][2018-11-0114:07:43][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Began transaction (1) for test context [DefaultTestContext@28261e8e testClass = UserTest, testInstance = mapper.UserTest@d737b89, testMethod = testselectbyprimarykey@UserTest, testException = [null], mergedContextConfiguration = [MergedContextConfiguration@8519cb4 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]; transaction manager [org.springframework.jdbc.datasource.DataSourceTransactionManager@35dab4eb]; rollback [true]
[INFO][2018-11-0114:07:43][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Rolled back transaction after test execution for test context [DefaultTestContext@28261e8e testClass = UserTest, testInstance = mapper.UserTest@d737b89, testMethod = testselectbyprimarykey@UserTest, testException = [null], mergedContextConfiguration = [MergedContextConfiguration@8519cb4 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]
[INFO][2018-11-0114:07:43][org.springframework.context.support.GenericApplicationContext]Closing org.springframework.context.support.GenericApplicationContext@5dfcfece: startup date [Thu Nov 01 14:07:41 CST 2018]; root of context hierarchy
[INFO][2018-11-0114:08:16][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [spring/spring-mybatis.xml]
[INFO][2018-11-0114:08:16][org.springframework.context.support.GenericApplicationContext]Refreshing org.springframework.context.support.GenericApplicationContext@5dfcfece: startup date [Thu Nov 01 14:08:16 CST 2018]; root of context hierarchy
[INFO][2018-11-0114:08:16][org.springframework.beans.factory.config.PropertyPlaceholderConfigurer]Loading properties file from class path resource [properties/jdbc.properties]
[INFO][2018-11-0114:08:16][org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor]JSR-330 'javax.inject.Inject' annotation found and supported for autowiring
[INFO][2018-11-0114:08:17][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Began transaction (1) for test context [DefaultTestContext@28261e8e testClass = UserTest, testInstance = mapper.UserTest@d737b89, testMethod = testselectbyprimarykey@UserTest, testException = [null], mergedContextConfiguration = [MergedContextConfiguration@8519cb4 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]; transaction manager [org.springframework.jdbc.datasource.DataSourceTransactionManager@35dab4eb]; rollback [true]
[INFO][2018-11-0114:08:17][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Rolled back transaction after test execution for test context [DefaultTestContext@28261e8e testClass = UserTest, testInstance = mapper.UserTest@d737b89, testMethod = testselectbyprimarykey@UserTest, testException = [null], mergedContextConfiguration = [MergedContextConfiguration@8519cb4 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]
[INFO][2018-11-0114:08:17][org.springframework.context.support.GenericApplicationContext]Closing org.springframework.context.support.GenericApplicationContext@5dfcfece: startup date [Thu Nov 01 14:08:16 CST 2018]; root of context hierarchy
[INFO][2018-11-0114:08:32][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [spring/spring-mybatis.xml]
[INFO][2018-11-0114:08:33][org.springframework.context.support.GenericApplicationContext]Refreshing org.springframework.context.support.GenericApplicationContext@5dfcfece: startup date [Thu Nov 01 14:08:32 CST 2018]; root of context hierarchy
[INFO][2018-11-0114:08:33][org.springframework.beans.factory.config.PropertyPlaceholderConfigurer]Loading properties file from class path resource [properties/jdbc.properties]
[INFO][2018-11-0114:08:33][org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor]JSR-330 'javax.inject.Inject' annotation found and supported for autowiring
[INFO][2018-11-0114:08:34][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Began transaction (1) for test context [DefaultTestContext@28261e8e testClass = UserTest, testInstance = mapper.UserTest@d737b89, testMethod = testselectbyprimarykey@UserTest, testException = [null], mergedContextConfiguration = [MergedContextConfiguration@8519cb4 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]; transaction manager [org.springframework.jdbc.datasource.DataSourceTransactionManager@35dab4eb]; rollback [true]
[INFO][2018-11-0114:08:34][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Rolled back transaction after test execution for test context [DefaultTestContext@28261e8e testClass = UserTest, testInstance = mapper.UserTest@d737b89, testMethod = testselectbyprimarykey@UserTest, testException = [null], mergedContextConfiguration = [MergedContextConfiguration@8519cb4 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]
[INFO][2018-11-0114:08:34][org.springframework.context.support.GenericApplicationContext]Closing org.springframework.context.support.GenericApplicationContext@5dfcfece: startup date [Thu Nov 01 14:08:32 CST 2018]; root of context hierarchy
[INFO][2018-12-0312:28:59][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [spring/spring-mybatis.xml]
[INFO][2018-12-0312:28:59][org.springframework.context.support.GenericApplicationContext]Refreshing org.springframework.context.support.GenericApplicationContext@5dfcfece: startup date [Mon Dec 03 12:28:59 CST 2018]; root of context hierarchy
[WARN][2018-12-0312:28:59][org.mybatis.spring.mapper.ClassPathMapperScanner]No MyBatis mapper was found in '[com.mayousheng.www.mapper]' package. Please check your configuration.
[INFO][2018-12-0312:28:59][org.springframework.beans.factory.config.PropertyPlaceholderConfigurer]Loading properties file from class path resource [properties/jdbc.properties]
[INFO][2018-12-0312:28:59][org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor]JSR-330 'javax.inject.Inject' annotation found and supported for autowiring
[ERROR][2018-12-0312:29:00][org.springframework.test.context.TestContextManager]Caught exception while allowing TestExecutionListener [org.springframework.test.context.support.DependencyInjectionTestExecutionListener@6c779568] to prepare test instance [mapper.UserTest@f381794]
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'mapper.UserTest': Injection of autowired dependencies failed; nested exception is org.springframework.beans.factory.BeanCreationException: Could not autowire field: private com.chwangteng.www.mapper.LaboratoryMapper mapper.UserTest.labMapper; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [com.chwangteng.www.mapper.LaboratoryMapper] found for dependency: expected at least 1 bean which qualifies as autowire candidate for this dependency. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessPropertyValues(AutowiredAnnotationBeanPostProcessor.java:292)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1185)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.autowireBeanProperties(AbstractAutowireCapableBeanFactory.java:384)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.injectDependencies(DependencyInjectionTestExecutionListener.java:110)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.prepareTestInstance(DependencyInjectionTestExecutionListener.java:75)
	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:319)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.createTest(SpringJUnit4ClassRunner.java:212)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner$1.runReflectiveCall(SpringJUnit4ClassRunner.java:289)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.methodBlock(SpringJUnit4ClassRunner.java:291)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:232)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:89)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.springframework.test.context.junit4.statements.RunBeforeTestClassCallbacks.evaluate(RunBeforeTestClassCallbacks.java:61)
	at org.springframework.test.context.junit4.statements.RunAfterTestClassCallbacks.evaluate(RunAfterTestClassCallbacks.java:71)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.run(SpringJUnit4ClassRunner.java:175)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:68)
	at com.intellij.rt.execution.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:47)
	at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:242)
	at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:70)
Caused by: org.springframework.beans.factory.BeanCreationException: Could not autowire field: private com.chwangteng.www.mapper.LaboratoryMapper mapper.UserTest.labMapper; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [com.chwangteng.www.mapper.LaboratoryMapper] found for dependency: expected at least 1 bean which qualifies as autowire candidate for this dependency. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:508)
	at org.springframework.beans.factory.annotation.InjectionMetadata.inject(InjectionMetadata.java:87)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessPropertyValues(AutowiredAnnotationBeanPostProcessor.java:289)
	... 25 more
Caused by: org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [com.chwangteng.www.mapper.LaboratoryMapper] found for dependency: expected at least 1 bean which qualifies as autowire candidate for this dependency. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.raiseNoSuchBeanDefinitionException(DefaultListableBeanFactory.java:1100)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:960)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:855)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:480)
	... 27 more
[INFO][2018-12-0312:29:00][org.springframework.context.support.GenericApplicationContext]Closing org.springframework.context.support.GenericApplicationContext@5dfcfece: startup date [Mon Dec 03 12:28:59 CST 2018]; root of context hierarchy
[INFO][2018-12-0312:34:22][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [spring/spring-mybatis.xml]
[INFO][2018-12-0312:34:22][org.springframework.context.support.GenericApplicationContext]Refreshing org.springframework.context.support.GenericApplicationContext@15761df8: startup date [Mon Dec 03 12:34:22 CST 2018]; root of context hierarchy
[WARN][2018-12-0312:34:22][org.mybatis.spring.mapper.ClassPathMapperScanner]No MyBatis mapper was found in '[com.mayousheng.www.mapper]' package. Please check your configuration.
[INFO][2018-12-0312:34:22][org.springframework.beans.factory.config.PropertyPlaceholderConfigurer]Loading properties file from class path resource [properties/jdbc.properties]
[INFO][2018-12-0312:34:22][org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor]JSR-330 'javax.inject.Inject' annotation found and supported for autowiring
[ERROR][2018-12-0312:34:23][org.springframework.test.context.TestContextManager]Caught exception while allowing TestExecutionListener [org.springframework.test.context.support.DependencyInjectionTestExecutionListener@4da602fc] to prepare test instance [mapper.UserTest@2a8d39c4]
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'mapper.UserTest': Injection of autowired dependencies failed; nested exception is org.springframework.beans.factory.BeanCreationException: Could not autowire field: private com.chwangteng.www.mapper.LaboratoryMapper mapper.UserTest.labMapper; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [com.chwangteng.www.mapper.LaboratoryMapper] found for dependency: expected at least 1 bean which qualifies as autowire candidate for this dependency. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessPropertyValues(AutowiredAnnotationBeanPostProcessor.java:292)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1185)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.autowireBeanProperties(AbstractAutowireCapableBeanFactory.java:384)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.injectDependencies(DependencyInjectionTestExecutionListener.java:110)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.prepareTestInstance(DependencyInjectionTestExecutionListener.java:75)
	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:319)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.createTest(SpringJUnit4ClassRunner.java:212)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner$1.runReflectiveCall(SpringJUnit4ClassRunner.java:289)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.methodBlock(SpringJUnit4ClassRunner.java:291)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:232)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:89)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.springframework.test.context.junit4.statements.RunBeforeTestClassCallbacks.evaluate(RunBeforeTestClassCallbacks.java:61)
	at org.springframework.test.context.junit4.statements.RunAfterTestClassCallbacks.evaluate(RunAfterTestClassCallbacks.java:71)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.run(SpringJUnit4ClassRunner.java:175)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:68)
	at com.intellij.rt.execution.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:47)
	at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:242)
	at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:70)
Caused by: org.springframework.beans.factory.BeanCreationException: Could not autowire field: private com.chwangteng.www.mapper.LaboratoryMapper mapper.UserTest.labMapper; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [com.chwangteng.www.mapper.LaboratoryMapper] found for dependency: expected at least 1 bean which qualifies as autowire candidate for this dependency. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:508)
	at org.springframework.beans.factory.annotation.InjectionMetadata.inject(InjectionMetadata.java:87)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessPropertyValues(AutowiredAnnotationBeanPostProcessor.java:289)
	... 25 more
Caused by: org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [com.chwangteng.www.mapper.LaboratoryMapper] found for dependency: expected at least 1 bean which qualifies as autowire candidate for this dependency. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.raiseNoSuchBeanDefinitionException(DefaultListableBeanFactory.java:1100)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:960)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:855)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:480)
	... 27 more
[INFO][2018-12-0312:34:23][org.springframework.context.support.GenericApplicationContext]Closing org.springframework.context.support.GenericApplicationContext@15761df8: startup date [Mon Dec 03 12:34:22 CST 2018]; root of context hierarchy
[INFO][2018-12-0312:34:46][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [spring/spring-mybatis.xml]
[INFO][2018-12-0312:34:46][org.springframework.context.support.GenericApplicationContext]Refreshing org.springframework.context.support.GenericApplicationContext@5dfcfece: startup date [Mon Dec 03 12:34:46 CST 2018]; root of context hierarchy
[WARN][2018-12-0312:34:46][org.mybatis.spring.mapper.ClassPathMapperScanner]No MyBatis mapper was found in '[com.mayousheng.www.mapper]' package. Please check your configuration.
[INFO][2018-12-0312:34:46][org.springframework.beans.factory.config.PropertyPlaceholderConfigurer]Loading properties file from class path resource [properties/jdbc.properties]
[INFO][2018-12-0312:34:46][org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor]JSR-330 'javax.inject.Inject' annotation found and supported for autowiring
[ERROR][2018-12-0312:34:47][org.springframework.test.context.TestContextManager]Caught exception while allowing TestExecutionListener [org.springframework.test.context.support.DependencyInjectionTestExecutionListener@6c779568] to prepare test instance [mapper.UserTest@f381794]
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'mapper.UserTest': Injection of autowired dependencies failed; nested exception is org.springframework.beans.factory.BeanCreationException: Could not autowire field: private com.chwangteng.www.mapper.LaboratoryMapper mapper.UserTest.labMapper; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [com.chwangteng.www.mapper.LaboratoryMapper] found for dependency: expected at least 1 bean which qualifies as autowire candidate for this dependency. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessPropertyValues(AutowiredAnnotationBeanPostProcessor.java:292)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1185)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.autowireBeanProperties(AbstractAutowireCapableBeanFactory.java:384)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.injectDependencies(DependencyInjectionTestExecutionListener.java:110)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.prepareTestInstance(DependencyInjectionTestExecutionListener.java:75)
	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:319)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.createTest(SpringJUnit4ClassRunner.java:212)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner$1.runReflectiveCall(SpringJUnit4ClassRunner.java:289)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.methodBlock(SpringJUnit4ClassRunner.java:291)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:232)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:89)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.springframework.test.context.junit4.statements.RunBeforeTestClassCallbacks.evaluate(RunBeforeTestClassCallbacks.java:61)
	at org.springframework.test.context.junit4.statements.RunAfterTestClassCallbacks.evaluate(RunAfterTestClassCallbacks.java:71)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.run(SpringJUnit4ClassRunner.java:175)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:68)
	at com.intellij.rt.execution.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:47)
	at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:242)
	at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:70)
Caused by: org.springframework.beans.factory.BeanCreationException: Could not autowire field: private com.chwangteng.www.mapper.LaboratoryMapper mapper.UserTest.labMapper; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [com.chwangteng.www.mapper.LaboratoryMapper] found for dependency: expected at least 1 bean which qualifies as autowire candidate for this dependency. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:508)
	at org.springframework.beans.factory.annotation.InjectionMetadata.inject(InjectionMetadata.java:87)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessPropertyValues(AutowiredAnnotationBeanPostProcessor.java:289)
	... 25 more
Caused by: org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [com.chwangteng.www.mapper.LaboratoryMapper] found for dependency: expected at least 1 bean which qualifies as autowire candidate for this dependency. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.raiseNoSuchBeanDefinitionException(DefaultListableBeanFactory.java:1100)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:960)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:855)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:480)
	... 27 more
[INFO][2018-12-0312:34:47][org.springframework.context.support.GenericApplicationContext]Closing org.springframework.context.support.GenericApplicationContext@5dfcfece: startup date [Mon Dec 03 12:34:46 CST 2018]; root of context hierarchy
[INFO][2018-12-0312:38:49][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [spring/spring-mybatis.xml]
[INFO][2018-12-0312:38:49][org.springframework.context.support.GenericApplicationContext]Refreshing org.springframework.context.support.GenericApplicationContext@5dfcfece: startup date [Mon Dec 03 12:38:49 CST 2018]; root of context hierarchy
[WARN][2018-12-0312:38:49][org.mybatis.spring.mapper.ClassPathMapperScanner]No MyBatis mapper was found in '[com.mayousheng.www.mapper]' package. Please check your configuration.
[INFO][2018-12-0312:38:49][org.springframework.beans.factory.config.PropertyPlaceholderConfigurer]Loading properties file from class path resource [properties/jdbc.properties]
[INFO][2018-12-0312:38:49][org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor]JSR-330 'javax.inject.Inject' annotation found and supported for autowiring
[ERROR][2018-12-0312:38:50][org.springframework.test.context.TestContextManager]Caught exception while allowing TestExecutionListener [org.springframework.test.context.support.DependencyInjectionTestExecutionListener@6c779568] to prepare test instance [mapper.UserTest@f381794]
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'mapper.UserTest': Injection of autowired dependencies failed; nested exception is org.springframework.beans.factory.BeanCreationException: Could not autowire field: private com.chwangteng.www.mapper.StudentMapper mapper.UserTest.studentMapper; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [com.chwangteng.www.mapper.StudentMapper] found for dependency: expected at least 1 bean which qualifies as autowire candidate for this dependency. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessPropertyValues(AutowiredAnnotationBeanPostProcessor.java:292)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1185)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.autowireBeanProperties(AbstractAutowireCapableBeanFactory.java:384)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.injectDependencies(DependencyInjectionTestExecutionListener.java:110)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.prepareTestInstance(DependencyInjectionTestExecutionListener.java:75)
	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:319)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.createTest(SpringJUnit4ClassRunner.java:212)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner$1.runReflectiveCall(SpringJUnit4ClassRunner.java:289)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.methodBlock(SpringJUnit4ClassRunner.java:291)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:232)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:89)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.springframework.test.context.junit4.statements.RunBeforeTestClassCallbacks.evaluate(RunBeforeTestClassCallbacks.java:61)
	at org.springframework.test.context.junit4.statements.RunAfterTestClassCallbacks.evaluate(RunAfterTestClassCallbacks.java:71)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.run(SpringJUnit4ClassRunner.java:175)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:68)
	at com.intellij.rt.execution.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:47)
	at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:242)
	at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:70)
Caused by: org.springframework.beans.factory.BeanCreationException: Could not autowire field: private com.chwangteng.www.mapper.StudentMapper mapper.UserTest.studentMapper; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [com.chwangteng.www.mapper.StudentMapper] found for dependency: expected at least 1 bean which qualifies as autowire candidate for this dependency. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:508)
	at org.springframework.beans.factory.annotation.InjectionMetadata.inject(InjectionMetadata.java:87)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessPropertyValues(AutowiredAnnotationBeanPostProcessor.java:289)
	... 25 more
Caused by: org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [com.chwangteng.www.mapper.StudentMapper] found for dependency: expected at least 1 bean which qualifies as autowire candidate for this dependency. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.raiseNoSuchBeanDefinitionException(DefaultListableBeanFactory.java:1100)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:960)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:855)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:480)
	... 27 more
[INFO][2018-12-0312:38:50][org.springframework.context.support.GenericApplicationContext]Closing org.springframework.context.support.GenericApplicationContext@5dfcfece: startup date [Mon Dec 03 12:38:49 CST 2018]; root of context hierarchy
[INFO][2018-12-0312:41:02][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [spring/spring-mybatis.xml]
[INFO][2018-12-0312:41:02][org.springframework.context.support.GenericApplicationContext]Refreshing org.springframework.context.support.GenericApplicationContext@5dfcfece: startup date [Mon Dec 03 12:41:02 CST 2018]; root of context hierarchy
[INFO][2018-12-0312:41:02][org.springframework.beans.factory.config.PropertyPlaceholderConfigurer]Loading properties file from class path resource [properties/jdbc.properties]
[INFO][2018-12-0312:41:02][org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor]JSR-330 'javax.inject.Inject' annotation found and supported for autowiring
[ERROR][2018-12-0312:41:03][org.springframework.test.context.TestContextManager]Caught exception while allowing TestExecutionListener [org.springframework.test.context.support.DependencyInjectionTestExecutionListener@483f6d77] to prepare test instance [mapper.UserTest@7e5afaa6]
java.lang.IllegalStateException: Failed to load ApplicationContext
	at org.springframework.test.context.CacheAwareContextLoaderDelegate.loadContext(CacheAwareContextLoaderDelegate.java:99)
	at org.springframework.test.context.DefaultTestContext.getApplicationContext(DefaultTestContext.java:101)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.injectDependencies(DependencyInjectionTestExecutionListener.java:109)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.prepareTestInstance(DependencyInjectionTestExecutionListener.java:75)
	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:319)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.createTest(SpringJUnit4ClassRunner.java:212)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner$1.runReflectiveCall(SpringJUnit4ClassRunner.java:289)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.methodBlock(SpringJUnit4ClassRunner.java:291)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:232)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:89)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.springframework.test.context.junit4.statements.RunBeforeTestClassCallbacks.evaluate(RunBeforeTestClassCallbacks.java:61)
	at org.springframework.test.context.junit4.statements.RunAfterTestClassCallbacks.evaluate(RunAfterTestClassCallbacks.java:71)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.run(SpringJUnit4ClassRunner.java:175)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:68)
	at com.intellij.rt.execution.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:47)
	at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:242)
	at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:70)
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'reportController': Injection of autowired dependencies failed; nested exception is org.springframework.beans.factory.BeanCreationException: Could not autowire field: private com.chwangteng.www.service.ReportService com.chwangteng.www.controller.ReportController.reportService; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [com.chwangteng.www.service.ReportService] found for dependency: expected at least 1 bean which qualifies as autowire candidate for this dependency. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessPropertyValues(AutowiredAnnotationBeanPostProcessor.java:292)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1185)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:537)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:475)
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:304)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:228)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:300)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:195)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:700)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:760)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:482)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:121)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:60)
	at org.springframework.test.context.support.AbstractDelegatingSmartContextLoader.delegateLoading(AbstractDelegatingSmartContextLoader.java:100)
	at org.springframework.test.context.support.AbstractDelegatingSmartContextLoader.loadContext(AbstractDelegatingSmartContextLoader.java:250)
	at org.springframework.test.context.CacheAwareContextLoaderDelegate.loadContextInternal(CacheAwareContextLoaderDelegate.java:64)
	at org.springframework.test.context.CacheAwareContextLoaderDelegate.loadContext(CacheAwareContextLoaderDelegate.java:91)
	... 24 more
Caused by: org.springframework.beans.factory.BeanCreationException: Could not autowire field: private com.chwangteng.www.service.ReportService com.chwangteng.www.controller.ReportController.reportService; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [com.chwangteng.www.service.ReportService] found for dependency: expected at least 1 bean which qualifies as autowire candidate for this dependency. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:508)
	at org.springframework.beans.factory.annotation.InjectionMetadata.inject(InjectionMetadata.java:87)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessPropertyValues(AutowiredAnnotationBeanPostProcessor.java:289)
	... 40 more
Caused by: org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [com.chwangteng.www.service.ReportService] found for dependency: expected at least 1 bean which qualifies as autowire candidate for this dependency. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.raiseNoSuchBeanDefinitionException(DefaultListableBeanFactory.java:1100)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:960)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:855)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:480)
	... 42 more
[INFO][2018-12-0312:41:22][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [spring/spring-mybatis.xml]
[INFO][2018-12-0312:41:23][org.springframework.context.support.GenericApplicationContext]Refreshing org.springframework.context.support.GenericApplicationContext@5dfcfece: startup date [Mon Dec 03 12:41:23 CST 2018]; root of context hierarchy
[INFO][2018-12-0312:41:23][org.springframework.beans.factory.config.PropertyPlaceholderConfigurer]Loading properties file from class path resource [properties/jdbc.properties]
[INFO][2018-12-0312:41:23][org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor]JSR-330 'javax.inject.Inject' annotation found and supported for autowiring
[ERROR][2018-12-0312:41:23][org.springframework.test.context.TestContextManager]Caught exception while allowing TestExecutionListener [org.springframework.test.context.support.DependencyInjectionTestExecutionListener@483f6d77] to prepare test instance [mapper.UserTest@7e5afaa6]
java.lang.IllegalStateException: Failed to load ApplicationContext
	at org.springframework.test.context.CacheAwareContextLoaderDelegate.loadContext(CacheAwareContextLoaderDelegate.java:99)
	at org.springframework.test.context.DefaultTestContext.getApplicationContext(DefaultTestContext.java:101)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.injectDependencies(DependencyInjectionTestExecutionListener.java:109)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.prepareTestInstance(DependencyInjectionTestExecutionListener.java:75)
	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:319)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.createTest(SpringJUnit4ClassRunner.java:212)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner$1.runReflectiveCall(SpringJUnit4ClassRunner.java:289)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.methodBlock(SpringJUnit4ClassRunner.java:291)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:232)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:89)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.springframework.test.context.junit4.statements.RunBeforeTestClassCallbacks.evaluate(RunBeforeTestClassCallbacks.java:61)
	at org.springframework.test.context.junit4.statements.RunAfterTestClassCallbacks.evaluate(RunAfterTestClassCallbacks.java:71)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.run(SpringJUnit4ClassRunner.java:175)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:68)
	at com.intellij.rt.execution.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:47)
	at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:242)
	at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:70)
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'reportController': Injection of autowired dependencies failed; nested exception is org.springframework.beans.factory.BeanCreationException: Could not autowire field: private com.chwangteng.www.service.ReportService com.chwangteng.www.controller.ReportController.reportService; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [com.chwangteng.www.service.ReportService] found for dependency: expected at least 1 bean which qualifies as autowire candidate for this dependency. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessPropertyValues(AutowiredAnnotationBeanPostProcessor.java:292)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1185)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:537)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:475)
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:304)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:228)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:300)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:195)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:700)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:760)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:482)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:121)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:60)
	at org.springframework.test.context.support.AbstractDelegatingSmartContextLoader.delegateLoading(AbstractDelegatingSmartContextLoader.java:100)
	at org.springframework.test.context.support.AbstractDelegatingSmartContextLoader.loadContext(AbstractDelegatingSmartContextLoader.java:250)
	at org.springframework.test.context.CacheAwareContextLoaderDelegate.loadContextInternal(CacheAwareContextLoaderDelegate.java:64)
	at org.springframework.test.context.CacheAwareContextLoaderDelegate.loadContext(CacheAwareContextLoaderDelegate.java:91)
	... 24 more
Caused by: org.springframework.beans.factory.BeanCreationException: Could not autowire field: private com.chwangteng.www.service.ReportService com.chwangteng.www.controller.ReportController.reportService; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [com.chwangteng.www.service.ReportService] found for dependency: expected at least 1 bean which qualifies as autowire candidate for this dependency. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:508)
	at org.springframework.beans.factory.annotation.InjectionMetadata.inject(InjectionMetadata.java:87)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessPropertyValues(AutowiredAnnotationBeanPostProcessor.java:289)
	... 40 more
Caused by: org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [com.chwangteng.www.service.ReportService] found for dependency: expected at least 1 bean which qualifies as autowire candidate for this dependency. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.raiseNoSuchBeanDefinitionException(DefaultListableBeanFactory.java:1100)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:960)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:855)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:480)
	... 42 more
[INFO][2018-12-0312:48:31][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [spring/spring-mybatis.xml]
[INFO][2018-12-0312:48:31][org.springframework.context.support.GenericApplicationContext]Refreshing org.springframework.context.support.GenericApplicationContext@5dfcfece: startup date [Mon Dec 03 12:48:31 CST 2018]; root of context hierarchy
[INFO][2018-12-0312:48:31][org.springframework.beans.factory.config.PropertyPlaceholderConfigurer]Loading properties file from class path resource [properties/jdbc.properties]
[INFO][2018-12-0312:48:31][org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor]JSR-330 'javax.inject.Inject' annotation found and supported for autowiring
[ERROR][2018-12-0312:48:32][org.springframework.test.context.TestContextManager]Caught exception while allowing TestExecutionListener [org.springframework.test.context.support.DependencyInjectionTestExecutionListener@483f6d77] to prepare test instance [mapper.UserTest@7e5afaa6]
java.lang.IllegalStateException: Failed to load ApplicationContext
	at org.springframework.test.context.CacheAwareContextLoaderDelegate.loadContext(CacheAwareContextLoaderDelegate.java:99)
	at org.springframework.test.context.DefaultTestContext.getApplicationContext(DefaultTestContext.java:101)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.injectDependencies(DependencyInjectionTestExecutionListener.java:109)
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.prepareTestInstance(DependencyInjectionTestExecutionListener.java:75)
	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:319)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.createTest(SpringJUnit4ClassRunner.java:212)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner$1.runReflectiveCall(SpringJUnit4ClassRunner.java:289)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.methodBlock(SpringJUnit4ClassRunner.java:291)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:232)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:89)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.springframework.test.context.junit4.statements.RunBeforeTestClassCallbacks.evaluate(RunBeforeTestClassCallbacks.java:61)
	at org.springframework.test.context.junit4.statements.RunAfterTestClassCallbacks.evaluate(RunAfterTestClassCallbacks.java:71)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.run(SpringJUnit4ClassRunner.java:175)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:68)
	at com.intellij.rt.execution.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:47)
	at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:242)
	at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:70)
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'reportController': Injection of autowired dependencies failed; nested exception is org.springframework.beans.factory.BeanCreationException: Could not autowire field: private com.chwangteng.www.service.ReportService com.chwangteng.www.controller.ReportController.reportService; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [com.chwangteng.www.service.ReportService] found for dependency: expected at least 1 bean which qualifies as autowire candidate for this dependency. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessPropertyValues(AutowiredAnnotationBeanPostProcessor.java:292)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1185)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:537)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:475)
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:304)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:228)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:300)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:195)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:700)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:760)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:482)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:121)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:60)
	at org.springframework.test.context.support.AbstractDelegatingSmartContextLoader.delegateLoading(AbstractDelegatingSmartContextLoader.java:100)
	at org.springframework.test.context.support.AbstractDelegatingSmartContextLoader.loadContext(AbstractDelegatingSmartContextLoader.java:250)
	at org.springframework.test.context.CacheAwareContextLoaderDelegate.loadContextInternal(CacheAwareContextLoaderDelegate.java:64)
	at org.springframework.test.context.CacheAwareContextLoaderDelegate.loadContext(CacheAwareContextLoaderDelegate.java:91)
	... 24 more
Caused by: org.springframework.beans.factory.BeanCreationException: Could not autowire field: private com.chwangteng.www.service.ReportService com.chwangteng.www.controller.ReportController.reportService; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [com.chwangteng.www.service.ReportService] found for dependency: expected at least 1 bean which qualifies as autowire candidate for this dependency. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:508)
	at org.springframework.beans.factory.annotation.InjectionMetadata.inject(InjectionMetadata.java:87)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessPropertyValues(AutowiredAnnotationBeanPostProcessor.java:289)
	... 40 more
Caused by: org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [com.chwangteng.www.service.ReportService] found for dependency: expected at least 1 bean which qualifies as autowire candidate for this dependency. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.raiseNoSuchBeanDefinitionException(DefaultListableBeanFactory.java:1100)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:960)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:855)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:480)
	... 42 more
[INFO][2018-12-0312:57:30][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [spring/spring-mybatis.xml]
[INFO][2018-12-0312:57:30][org.springframework.context.support.GenericApplicationContext]Refreshing org.springframework.context.support.GenericApplicationContext@5dfcfece: startup date [Mon Dec 03 12:57:30 CST 2018]; root of context hierarchy
[INFO][2018-12-0312:57:31][org.springframework.beans.factory.config.PropertyPlaceholderConfigurer]Loading properties file from class path resource [properties/jdbc.properties]
[INFO][2018-12-0312:57:31][org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor]JSR-330 'javax.inject.Inject' annotation found and supported for autowiring
[INFO][2018-12-0312:57:32][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Began transaction (1) for test context [DefaultTestContext@268f106e testClass = UserTest, testInstance = mapper.UserTest@6e9a5ed8, testMethod = testceratecriteria@UserTest, testException = [null], mergedContextConfiguration = [MergedContextConfiguration@7e057f43 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]; transaction manager [org.springframework.jdbc.datasource.DataSourceTransactionManager@6c284af]; rollback [true]
[INFO][2018-12-0312:57:32][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Rolled back transaction after test execution for test context [DefaultTestContext@268f106e testClass = UserTest, testInstance = mapper.UserTest@6e9a5ed8, testMethod = testceratecriteria@UserTest, testException = [null], mergedContextConfiguration = [MergedContextConfiguration@7e057f43 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]
[INFO][2018-12-0312:57:32][org.springframework.context.support.GenericApplicationContext]Closing org.springframework.context.support.GenericApplicationContext@5dfcfece: startup date [Mon Dec 03 12:57:30 CST 2018]; root of context hierarchy
[INFO][2018-12-0313:03:39][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [spring/spring-mybatis.xml]
[INFO][2018-12-0313:03:39][org.springframework.context.support.GenericApplicationContext]Refreshing org.springframework.context.support.GenericApplicationContext@5dfcfece: startup date [Mon Dec 03 13:03:39 CST 2018]; root of context hierarchy
[INFO][2018-12-0313:03:40][org.springframework.beans.factory.config.PropertyPlaceholderConfigurer]Loading properties file from class path resource [properties/jdbc.properties]
[INFO][2018-12-0313:03:40][org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor]JSR-330 'javax.inject.Inject' annotation found and supported for autowiring
[INFO][2018-12-0313:03:41][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Began transaction (1) for test context [DefaultTestContext@268f106e testClass = UserTest, testInstance = mapper.UserTest@6e9a5ed8, testMethod = testselectbyprimarykey@UserTest, testException = [null], mergedContextConfiguration = [MergedContextConfiguration@7e057f43 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]; transaction manager [org.springframework.jdbc.datasource.DataSourceTransactionManager@6c284af]; rollback [true]
[INFO][2018-12-0313:03:41][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Rolled back transaction after test execution for test context [DefaultTestContext@268f106e testClass = UserTest, testInstance = mapper.UserTest@6e9a5ed8, testMethod = testselectbyprimarykey@UserTest, testException = [null], mergedContextConfiguration = [MergedContextConfiguration@7e057f43 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]
[INFO][2018-12-0313:03:41][org.springframework.context.support.GenericApplicationContext]Closing org.springframework.context.support.GenericApplicationContext@5dfcfece: startup date [Mon Dec 03 13:03:39 CST 2018]; root of context hierarchy
[INFO][2018-12-0515:11:48][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [spring/spring-mybatis.xml]
[INFO][2018-12-0515:11:48][org.springframework.context.support.GenericApplicationContext]Refreshing org.springframework.context.support.GenericApplicationContext@5dfcfece: startup date [Wed Dec 05 15:11:48 CST 2018]; root of context hierarchy
[INFO][2018-12-0515:11:48][org.springframework.beans.factory.config.PropertyPlaceholderConfigurer]Loading properties file from class path resource [properties/jdbc.properties]
[INFO][2018-12-0515:11:48][org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor]JSR-330 'javax.inject.Inject' annotation found and supported for autowiring
[INFO][2018-12-0515:11:50][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Began transaction (1) for test context [DefaultTestContext@76494737 testClass = UserTest, testInstance = mapper.UserTest@4a003cbe, testMethod = getreport@UserTest, testException = [null], mergedContextConfiguration = [MergedContextConfiguration@4082ba93 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]; transaction manager [org.springframework.jdbc.datasource.DataSourceTransactionManager@17fc391b]; rollback [true]
[INFO][2018-12-0515:11:50][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Rolled back transaction after test execution for test context [DefaultTestContext@76494737 testClass = UserTest, testInstance = mapper.UserTest@4a003cbe, testMethod = getreport@UserTest, testException = org.mybatis.spring.MyBatisSystemException: nested exception is org.apache.ibatis.reflection.ReflectionException: Error instantiating class com.chwangteng.www.pojo.ReportWithBLOBs with invalid types (Integer,String,Integer,Integer,Integer,Date,Date,String,String,String,String,) or values (1,王腾第8周,1,0,null,null,null,null,null,null,null,). Cause: java.lang.NoSuchMethodException: com.chwangteng.www.pojo.ReportWithBLOBs.<init>(java.lang.Integer, java.lang.String, java.lang.Integer, java.lang.Integer, java.lang.Integer, java.util.Date, java.util.Date, java.lang.String, java.lang.String, java.lang.String, java.lang.String), mergedContextConfiguration = [MergedContextConfiguration@4082ba93 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]
[INFO][2018-12-0515:11:50][org.springframework.context.support.GenericApplicationContext]Closing org.springframework.context.support.GenericApplicationContext@5dfcfece: startup date [Wed Dec 05 15:11:48 CST 2018]; root of context hierarchy
[INFO][2018-12-0515:34:41][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [spring/spring-mybatis.xml]
[INFO][2018-12-0515:34:41][org.springframework.context.support.GenericApplicationContext]Refreshing org.springframework.context.support.GenericApplicationContext@5dfcfece: startup date [Wed Dec 05 15:34:41 CST 2018]; root of context hierarchy
[INFO][2018-12-0515:34:42][org.springframework.beans.factory.config.PropertyPlaceholderConfigurer]Loading properties file from class path resource [properties/jdbc.properties]
[INFO][2018-12-0515:34:42][org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor]JSR-330 'javax.inject.Inject' annotation found and supported for autowiring
[INFO][2018-12-0515:34:43][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Began transaction (1) for test context [DefaultTestContext@76494737 testClass = UserTest, testInstance = mapper.UserTest@4a003cbe, testMethod = getreport@UserTest, testException = [null], mergedContextConfiguration = [MergedContextConfiguration@4082ba93 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]; transaction manager [org.springframework.jdbc.datasource.DataSourceTransactionManager@17fc391b]; rollback [true]
[INFO][2018-12-0515:34:43][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Rolled back transaction after test execution for test context [DefaultTestContext@76494737 testClass = UserTest, testInstance = mapper.UserTest@4a003cbe, testMethod = getreport@UserTest, testException = org.mybatis.spring.MyBatisSystemException: nested exception is org.apache.ibatis.reflection.ReflectionException: Error instantiating class com.chwangteng.www.pojo.ReportWithBLOBs with invalid types (Integer,String,Integer,Integer,Integer,Date,Date,String,String,String,String,) or values (1,王腾第8周,1,0,null,null,null,null,null,null,null,). Cause: java.lang.NoSuchMethodException: com.chwangteng.www.pojo.ReportWithBLOBs.<init>(java.lang.Integer, java.lang.String, java.lang.Integer, java.lang.Integer, java.lang.Integer, java.util.Date, java.util.Date, java.lang.String, java.lang.String, java.lang.String, java.lang.String), mergedContextConfiguration = [MergedContextConfiguration@4082ba93 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]
[INFO][2018-12-0515:34:43][org.springframework.context.support.GenericApplicationContext]Closing org.springframework.context.support.GenericApplicationContext@5dfcfece: startup date [Wed Dec 05 15:34:41 CST 2018]; root of context hierarchy
[INFO][2018-12-0515:35:18][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [spring/spring-mybatis.xml]
[INFO][2018-12-0515:35:18][org.springframework.context.support.GenericApplicationContext]Refreshing org.springframework.context.support.GenericApplicationContext@5dfcfece: startup date [Wed Dec 05 15:35:18 CST 2018]; root of context hierarchy
[INFO][2018-12-0515:35:19][org.springframework.beans.factory.config.PropertyPlaceholderConfigurer]Loading properties file from class path resource [properties/jdbc.properties]
[INFO][2018-12-0515:35:19][org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor]JSR-330 'javax.inject.Inject' annotation found and supported for autowiring
[INFO][2018-12-0515:35:20][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Began transaction (1) for test context [DefaultTestContext@76494737 testClass = UserTest, testInstance = mapper.UserTest@4a003cbe, testMethod = getreport@UserTest, testException = [null], mergedContextConfiguration = [MergedContextConfiguration@4082ba93 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]; transaction manager [org.springframework.jdbc.datasource.DataSourceTransactionManager@17fc391b]; rollback [true]
[INFO][2018-12-0515:35:20][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Rolled back transaction after test execution for test context [DefaultTestContext@76494737 testClass = UserTest, testInstance = mapper.UserTest@4a003cbe, testMethod = getreport@UserTest, testException = org.mybatis.spring.MyBatisSystemException: nested exception is org.apache.ibatis.reflection.ReflectionException: Error instantiating class com.chwangteng.www.pojo.ReportWithBLOBs with invalid types (Integer,String,Integer,Integer,Integer,Date,Date,String,String,String,String,) or values (1,王腾第8周,1,0,99,Wed Dec 05 15:34:15 CST 2018,Wed Dec 05 15:35:08 CST 2018,豆腐干反对大哥,梵蒂冈的是,覆盖,hello,). Cause: java.lang.NoSuchMethodException: com.chwangteng.www.pojo.ReportWithBLOBs.<init>(java.lang.Integer, java.lang.String, java.lang.Integer, java.lang.Integer, java.lang.Integer, java.util.Date, java.util.Date, java.lang.String, java.lang.String, java.lang.String, java.lang.String), mergedContextConfiguration = [MergedContextConfiguration@4082ba93 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]
[INFO][2018-12-0515:35:20][org.springframework.context.support.GenericApplicationContext]Closing org.springframework.context.support.GenericApplicationContext@5dfcfece: startup date [Wed Dec 05 15:35:18 CST 2018]; root of context hierarchy
[INFO][2018-12-0515:37:26][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [spring/spring-mybatis.xml]
[INFO][2018-12-0515:37:26][org.springframework.context.support.GenericApplicationContext]Refreshing org.springframework.context.support.GenericApplicationContext@5dfcfece: startup date [Wed Dec 05 15:37:26 CST 2018]; root of context hierarchy
[INFO][2018-12-0515:37:26][org.springframework.beans.factory.config.PropertyPlaceholderConfigurer]Loading properties file from class path resource [properties/jdbc.properties]
[INFO][2018-12-0515:37:26][org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor]JSR-330 'javax.inject.Inject' annotation found and supported for autowiring
[INFO][2018-12-0515:37:28][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Began transaction (1) for test context [DefaultTestContext@76494737 testClass = UserTest, testInstance = mapper.UserTest@4a003cbe, testMethod = getreport@UserTest, testException = [null], mergedContextConfiguration = [MergedContextConfiguration@4082ba93 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]; transaction manager [org.springframework.jdbc.datasource.DataSourceTransactionManager@17fc391b]; rollback [true]
[INFO][2018-12-0515:37:28][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Rolled back transaction after test execution for test context [DefaultTestContext@76494737 testClass = UserTest, testInstance = mapper.UserTest@4a003cbe, testMethod = getreport@UserTest, testException = org.mybatis.spring.MyBatisSystemException: nested exception is org.apache.ibatis.reflection.ReflectionException: Error instantiating class com.chwangteng.www.pojo.ReportWithBLOBs with invalid types (Integer,String,Integer,Integer,Integer,Date,Date,String,String,String,String,) or values (1,王腾第8周,1,0,99,Wed Dec 05 15:34:15 CST 2018,Wed Dec 05 15:35:08 CST 2018,豆腐干反对大哥,梵蒂冈的是,覆盖,hello,). Cause: java.lang.NoSuchMethodException: com.chwangteng.www.pojo.ReportWithBLOBs.<init>(java.lang.Integer, java.lang.String, java.lang.Integer, java.lang.Integer, java.lang.Integer, java.util.Date, java.util.Date, java.lang.String, java.lang.String, java.lang.String, java.lang.String), mergedContextConfiguration = [MergedContextConfiguration@4082ba93 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]
[INFO][2018-12-0515:37:28][org.springframework.context.support.GenericApplicationContext]Closing org.springframework.context.support.GenericApplicationContext@5dfcfece: startup date [Wed Dec 05 15:37:26 CST 2018]; root of context hierarchy
[INFO][2018-12-0515:40:49][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [spring/spring-mybatis.xml]
[INFO][2018-12-0515:40:50][org.springframework.context.support.GenericApplicationContext]Refreshing org.springframework.context.support.GenericApplicationContext@5dfcfece: startup date [Wed Dec 05 15:40:50 CST 2018]; root of context hierarchy
[INFO][2018-12-0515:40:50][org.springframework.beans.factory.config.PropertyPlaceholderConfigurer]Loading properties file from class path resource [properties/jdbc.properties]
[INFO][2018-12-0515:40:50][org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor]JSR-330 'javax.inject.Inject' annotation found and supported for autowiring
[INFO][2018-12-0515:40:51][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Began transaction (1) for test context [DefaultTestContext@76494737 testClass = UserTest, testInstance = mapper.UserTest@4a003cbe, testMethod = getreport@UserTest, testException = [null], mergedContextConfiguration = [MergedContextConfiguration@4082ba93 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]; transaction manager [org.springframework.jdbc.datasource.DataSourceTransactionManager@17fc391b]; rollback [true]
[INFO][2018-12-0515:40:51][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Rolled back transaction after test execution for test context [DefaultTestContext@76494737 testClass = UserTest, testInstance = mapper.UserTest@4a003cbe, testMethod = getreport@UserTest, testException = org.mybatis.spring.MyBatisSystemException: nested exception is org.apache.ibatis.reflection.ReflectionException: Error instantiating class com.chwangteng.www.pojo.ReportWithBLOBs with invalid types (Integer,String,Integer,Integer,Integer,Date,Date,String,String,String,String,) or values (1,王腾第8周,1,0,99,Wed Dec 05 15:34:15 CST 2018,Wed Dec 05 15:35:08 CST 2018,豆腐干反对大哥,梵蒂冈的是,覆盖,hello,). Cause: java.lang.NoSuchMethodException: com.chwangteng.www.pojo.ReportWithBLOBs.<init>(java.lang.Integer, java.lang.String, java.lang.Integer, java.lang.Integer, java.lang.Integer, java.util.Date, java.util.Date, java.lang.String, java.lang.String, java.lang.String, java.lang.String), mergedContextConfiguration = [MergedContextConfiguration@4082ba93 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]
[INFO][2018-12-0515:40:51][org.springframework.context.support.GenericApplicationContext]Closing org.springframework.context.support.GenericApplicationContext@5dfcfece: startup date [Wed Dec 05 15:40:50 CST 2018]; root of context hierarchy
[INFO][2018-12-0515:45:36][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [spring/spring-mybatis.xml]
[INFO][2018-12-0515:45:37][org.springframework.context.support.GenericApplicationContext]Refreshing org.springframework.context.support.GenericApplicationContext@5dfcfece: startup date [Wed Dec 05 15:45:37 CST 2018]; root of context hierarchy
[INFO][2018-12-0515:45:37][org.springframework.beans.factory.config.PropertyPlaceholderConfigurer]Loading properties file from class path resource [properties/jdbc.properties]
[INFO][2018-12-0515:45:37][org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor]JSR-330 'javax.inject.Inject' annotation found and supported for autowiring
[INFO][2018-12-0515:45:38][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Began transaction (1) for test context [DefaultTestContext@76494737 testClass = UserTest, testInstance = mapper.UserTest@4a003cbe, testMethod = getreport@UserTest, testException = [null], mergedContextConfiguration = [MergedContextConfiguration@4082ba93 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]; transaction manager [org.springframework.jdbc.datasource.DataSourceTransactionManager@17fc391b]; rollback [true]
[INFO][2018-12-0515:45:38][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Rolled back transaction after test execution for test context [DefaultTestContext@76494737 testClass = UserTest, testInstance = mapper.UserTest@4a003cbe, testMethod = getreport@UserTest, testException = [null], mergedContextConfiguration = [MergedContextConfiguration@4082ba93 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]
[INFO][2018-12-0515:45:38][org.springframework.context.support.GenericApplicationContext]Closing org.springframework.context.support.GenericApplicationContext@5dfcfece: startup date [Wed Dec 05 15:45:37 CST 2018]; root of context hierarchy
[INFO][2018-12-0600:00:56][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [spring/spring-mybatis.xml]
[INFO][2018-12-0600:00:56][org.springframework.context.support.GenericApplicationContext]Refreshing org.springframework.context.support.GenericApplicationContext@5dfcfece: startup date [Thu Dec 06 00:00:56 CST 2018]; root of context hierarchy
[INFO][2018-12-0600:00:57][org.springframework.beans.factory.config.PropertyPlaceholderConfigurer]Loading properties file from class path resource [properties/jdbc.properties]
[INFO][2018-12-0600:00:57][org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor]JSR-330 'javax.inject.Inject' annotation found and supported for autowiring
[INFO][2018-12-0600:00:58][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Began transaction (1) for test context [DefaultTestContext@76494737 testClass = UserTest, testInstance = mapper.UserTest@4a003cbe, testMethod = testnoti@UserTest, testException = [null], mergedContextConfiguration = [MergedContextConfiguration@4082ba93 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]; transaction manager [org.springframework.jdbc.datasource.DataSourceTransactionManager@17fc391b]; rollback [true]
[INFO][2018-12-0600:00:58][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [org/springframework/jdbc/support/sql-error-codes.xml]
[INFO][2018-12-0600:00:58][org.springframework.jdbc.support.SQLErrorCodesFactory]SQLErrorCodes loaded: [DB2, Derby, H2, HSQL, Informix, MS-SQL, MySQL, Oracle, PostgreSQL, Sybase]
[INFO][2018-12-0600:00:58][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Rolled back transaction after test execution for test context [DefaultTestContext@76494737 testClass = UserTest, testInstance = mapper.UserTest@4a003cbe, testMethod = testnoti@UserTest, testException = org.springframework.jdbc.BadSqlGrammarException: 
### Error querying database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'read
   
    from notification
     
       
     WHERE (  user_id = 1
         ' at line 4
### The error may exist in file [C:\Users\chwangteng\Documents\GitHub\Report\target\classes\mybatis\mapper\NotificationMapper.xml]
### The error may involve com.chwangteng.www.mapper.NotificationMapper.selectByExample-Inline
### The error occurred while setting parameters
### SQL: select                 id, user_id, user_type, generate_time, title, content, read         from notification                    WHERE (  user_id = ?                                                                and user_type = ? )
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'read
   
    from notification
     
       
     WHERE (  user_id = 1
         ' at line 4
; bad SQL grammar []; nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'read
   
    from notification
     
       
     WHERE (  user_id = 1
         ' at line 4, mergedContextConfiguration = [MergedContextConfiguration@4082ba93 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]
[INFO][2018-12-0600:00:58][org.springframework.context.support.GenericApplicationContext]Closing org.springframework.context.support.GenericApplicationContext@5dfcfece: startup date [Thu Dec 06 00:00:56 CST 2018]; root of context hierarchy
[INFO][2018-12-0600:04:56][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [spring/spring-mybatis.xml]
[INFO][2018-12-0600:04:56][org.springframework.context.support.GenericApplicationContext]Refreshing org.springframework.context.support.GenericApplicationContext@5dfcfece: startup date [Thu Dec 06 00:04:56 CST 2018]; root of context hierarchy
[INFO][2018-12-0600:04:56][org.springframework.beans.factory.config.PropertyPlaceholderConfigurer]Loading properties file from class path resource [properties/jdbc.properties]
[INFO][2018-12-0600:04:56][org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor]JSR-330 'javax.inject.Inject' annotation found and supported for autowiring
[INFO][2018-12-0600:04:58][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Began transaction (1) for test context [DefaultTestContext@76494737 testClass = UserTest, testInstance = mapper.UserTest@4a003cbe, testMethod = testnoti@UserTest, testException = [null], mergedContextConfiguration = [MergedContextConfiguration@4082ba93 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]; transaction manager [org.springframework.jdbc.datasource.DataSourceTransactionManager@17fc391b]; rollback [true]
[INFO][2018-12-0600:04:58][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [org/springframework/jdbc/support/sql-error-codes.xml]
[INFO][2018-12-0600:04:58][org.springframework.jdbc.support.SQLErrorCodesFactory]SQLErrorCodes loaded: [DB2, Derby, H2, HSQL, Informix, MS-SQL, MySQL, Oracle, PostgreSQL, Sybase]
[INFO][2018-12-0600:04:58][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Rolled back transaction after test execution for test context [DefaultTestContext@76494737 testClass = UserTest, testInstance = mapper.UserTest@4a003cbe, testMethod = testnoti@UserTest, testException = org.springframework.jdbc.BadSqlGrammarException: 
### Error querying database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'read
   
    from notification
     
       
     WHERE (  user_id = 1
         ' at line 4
### The error may exist in file [C:\Users\chwangteng\Documents\GitHub\Report\target\classes\mybatis\mapper\NotificationMapper.xml]
### The error may involve com.chwangteng.www.mapper.NotificationMapper.selectByExample-Inline
### The error occurred while setting parameters
### SQL: select                 id, user_id, user_type, generate_time, title, content, read         from notification                    WHERE (  user_id = ?                                                                and user_type = ? )
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'read
   
    from notification
     
       
     WHERE (  user_id = 1
         ' at line 4
; bad SQL grammar []; nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'read
   
    from notification
     
       
     WHERE (  user_id = 1
         ' at line 4, mergedContextConfiguration = [MergedContextConfiguration@4082ba93 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]
[INFO][2018-12-0600:04:58][org.springframework.context.support.GenericApplicationContext]Closing org.springframework.context.support.GenericApplicationContext@5dfcfece: startup date [Thu Dec 06 00:04:56 CST 2018]; root of context hierarchy
[INFO][2018-12-0600:05:28][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [spring/spring-mybatis.xml]
[INFO][2018-12-0600:05:28][org.springframework.context.support.GenericApplicationContext]Refreshing org.springframework.context.support.GenericApplicationContext@5dfcfece: startup date [Thu Dec 06 00:05:28 CST 2018]; root of context hierarchy
[INFO][2018-12-0600:05:29][org.springframework.beans.factory.config.PropertyPlaceholderConfigurer]Loading properties file from class path resource [properties/jdbc.properties]
[INFO][2018-12-0600:05:29][org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor]JSR-330 'javax.inject.Inject' annotation found and supported for autowiring
[INFO][2018-12-0600:05:30][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Began transaction (1) for test context [DefaultTestContext@76494737 testClass = UserTest, testInstance = mapper.UserTest@4a003cbe, testMethod = testnoti@UserTest, testException = [null], mergedContextConfiguration = [MergedContextConfiguration@4082ba93 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]; transaction manager [org.springframework.jdbc.datasource.DataSourceTransactionManager@17fc391b]; rollback [true]
[INFO][2018-12-0600:05:30][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [org/springframework/jdbc/support/sql-error-codes.xml]
[INFO][2018-12-0600:05:30][org.springframework.jdbc.support.SQLErrorCodesFactory]SQLErrorCodes loaded: [DB2, Derby, H2, HSQL, Informix, MS-SQL, MySQL, Oracle, PostgreSQL, Sybase]
[INFO][2018-12-0600:05:30][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Rolled back transaction after test execution for test context [DefaultTestContext@76494737 testClass = UserTest, testInstance = mapper.UserTest@4a003cbe, testMethod = testnoti@UserTest, testException = org.springframework.jdbc.BadSqlGrammarException: 
### Error querying database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'read
   
    from notification
     
       
     WHERE (  user_id = 1 )' at line 4
### The error may exist in file [C:\Users\chwangteng\Documents\GitHub\Report\target\classes\mybatis\mapper\NotificationMapper.xml]
### The error may involve com.chwangteng.www.mapper.NotificationMapper.selectByExample-Inline
### The error occurred while setting parameters
### SQL: select                 id, user_id, user_type, generate_time, title, content, read         from notification                    WHERE (  user_id = ? )
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'read
   
    from notification
     
       
     WHERE (  user_id = 1 )' at line 4
; bad SQL grammar []; nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'read
   
    from notification
     
       
     WHERE (  user_id = 1 )' at line 4, mergedContextConfiguration = [MergedContextConfiguration@4082ba93 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]
[INFO][2018-12-0600:05:30][org.springframework.context.support.GenericApplicationContext]Closing org.springframework.context.support.GenericApplicationContext@5dfcfece: startup date [Thu Dec 06 00:05:28 CST 2018]; root of context hierarchy
[INFO][2018-12-0600:05:39][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [spring/spring-mybatis.xml]
[INFO][2018-12-0600:05:39][org.springframework.context.support.GenericApplicationContext]Refreshing org.springframework.context.support.GenericApplicationContext@5dfcfece: startup date [Thu Dec 06 00:05:39 CST 2018]; root of context hierarchy
[INFO][2018-12-0600:05:40][org.springframework.beans.factory.config.PropertyPlaceholderConfigurer]Loading properties file from class path resource [properties/jdbc.properties]
[INFO][2018-12-0600:05:40][org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor]JSR-330 'javax.inject.Inject' annotation found and supported for autowiring
[INFO][2018-12-0600:05:41][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Began transaction (1) for test context [DefaultTestContext@76494737 testClass = UserTest, testInstance = mapper.UserTest@4a003cbe, testMethod = testnoti@UserTest, testException = [null], mergedContextConfiguration = [MergedContextConfiguration@4082ba93 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]; transaction manager [org.springframework.jdbc.datasource.DataSourceTransactionManager@17fc391b]; rollback [true]
[INFO][2018-12-0600:05:41][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [org/springframework/jdbc/support/sql-error-codes.xml]
[INFO][2018-12-0600:05:41][org.springframework.jdbc.support.SQLErrorCodesFactory]SQLErrorCodes loaded: [DB2, Derby, H2, HSQL, Informix, MS-SQL, MySQL, Oracle, PostgreSQL, Sybase]
[INFO][2018-12-0600:05:41][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Rolled back transaction after test execution for test context [DefaultTestContext@76494737 testClass = UserTest, testInstance = mapper.UserTest@4a003cbe, testMethod = testnoti@UserTest, testException = org.springframework.jdbc.BadSqlGrammarException: 
### Error querying database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'read
   
    from notification' at line 4
### The error may exist in file [C:\Users\chwangteng\Documents\GitHub\Report\target\classes\mybatis\mapper\NotificationMapper.xml]
### The error may involve com.chwangteng.www.mapper.NotificationMapper.selectByExample-Inline
### The error occurred while setting parameters
### SQL: select                 id, user_id, user_type, generate_time, title, content, read         from notification
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'read
   
    from notification' at line 4
; bad SQL grammar []; nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'read
   
    from notification' at line 4, mergedContextConfiguration = [MergedContextConfiguration@4082ba93 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]
[INFO][2018-12-0600:05:41][org.springframework.context.support.GenericApplicationContext]Closing org.springframework.context.support.GenericApplicationContext@5dfcfece: startup date [Thu Dec 06 00:05:39 CST 2018]; root of context hierarchy
[INFO][2018-12-0600:06:20][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [spring/spring-mybatis.xml]
[INFO][2018-12-0600:06:21][org.springframework.context.support.GenericApplicationContext]Refreshing org.springframework.context.support.GenericApplicationContext@5dfcfece: startup date [Thu Dec 06 00:06:21 CST 2018]; root of context hierarchy
[INFO][2018-12-0600:06:21][org.springframework.beans.factory.config.PropertyPlaceholderConfigurer]Loading properties file from class path resource [properties/jdbc.properties]
[INFO][2018-12-0600:06:21][org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor]JSR-330 'javax.inject.Inject' annotation found and supported for autowiring
[INFO][2018-12-0600:06:22][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Began transaction (1) for test context [DefaultTestContext@76494737 testClass = UserTest, testInstance = mapper.UserTest@4a003cbe, testMethod = testnoti@UserTest, testException = [null], mergedContextConfiguration = [MergedContextConfiguration@4082ba93 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]; transaction manager [org.springframework.jdbc.datasource.DataSourceTransactionManager@17fc391b]; rollback [true]
[INFO][2018-12-0600:06:22][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [org/springframework/jdbc/support/sql-error-codes.xml]
[INFO][2018-12-0600:06:22][org.springframework.jdbc.support.SQLErrorCodesFactory]SQLErrorCodes loaded: [DB2, Derby, H2, HSQL, Informix, MS-SQL, MySQL, Oracle, PostgreSQL, Sybase]
[INFO][2018-12-0600:06:22][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Rolled back transaction after test execution for test context [DefaultTestContext@76494737 testClass = UserTest, testInstance = mapper.UserTest@4a003cbe, testMethod = testnoti@UserTest, testException = org.springframework.jdbc.BadSqlGrammarException: 
### Error querying database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'read
   
    from notification
    where id = 1' at line 3
### The error may exist in file [C:\Users\chwangteng\Documents\GitHub\Report\target\classes\mybatis\mapper\NotificationMapper.xml]
### The error may involve com.chwangteng.www.mapper.NotificationMapper.selectByPrimaryKey-Inline
### The error occurred while setting parameters
### SQL: select            id, user_id, user_type, generate_time, title, content, read         from notification     where id = ?
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'read
   
    from notification
    where id = 1' at line 3
; bad SQL grammar []; nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'read
   
    from notification
    where id = 1' at line 3, mergedContextConfiguration = [MergedContextConfiguration@4082ba93 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]
[INFO][2018-12-0600:06:22][org.springframework.context.support.GenericApplicationContext]Closing org.springframework.context.support.GenericApplicationContext@5dfcfece: startup date [Thu Dec 06 00:06:21 CST 2018]; root of context hierarchy
[INFO][2018-12-0600:10:36][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [spring/spring-mybatis.xml]
[INFO][2018-12-0600:10:36][org.springframework.context.support.GenericApplicationContext]Refreshing org.springframework.context.support.GenericApplicationContext@5dfcfece: startup date [Thu Dec 06 00:10:36 CST 2018]; root of context hierarchy
[INFO][2018-12-0600:10:37][org.springframework.beans.factory.config.PropertyPlaceholderConfigurer]Loading properties file from class path resource [properties/jdbc.properties]
[INFO][2018-12-0600:10:37][org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor]JSR-330 'javax.inject.Inject' annotation found and supported for autowiring
[INFO][2018-12-0600:10:38][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Began transaction (1) for test context [DefaultTestContext@76494737 testClass = UserTest, testInstance = mapper.UserTest@4a003cbe, testMethod = testnoti@UserTest, testException = [null], mergedContextConfiguration = [MergedContextConfiguration@4082ba93 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]; transaction manager [org.springframework.jdbc.datasource.DataSourceTransactionManager@17fc391b]; rollback [true]
[INFO][2018-12-0600:10:38][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [org/springframework/jdbc/support/sql-error-codes.xml]
[INFO][2018-12-0600:10:38][org.springframework.jdbc.support.SQLErrorCodesFactory]SQLErrorCodes loaded: [DB2, Derby, H2, HSQL, Informix, MS-SQL, MySQL, Oracle, PostgreSQL, Sybase]
[INFO][2018-12-0600:10:38][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Rolled back transaction after test execution for test context [DefaultTestContext@76494737 testClass = UserTest, testInstance = mapper.UserTest@4a003cbe, testMethod = testnoti@UserTest, testException = org.springframework.jdbc.BadSqlGrammarException: 
### Error querying database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'read
   
    from notification' at line 4
### The error may exist in file [C:\Users\chwangteng\Documents\GitHub\Report\target\classes\mybatis\mapper\NotificationMapper.xml]
### The error may involve com.chwangteng.www.mapper.NotificationMapper.selectByExample-Inline
### The error occurred while setting parameters
### SQL: select                 id, user_id, user_type, generate_time, title, content, read         from notification
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'read
   
    from notification' at line 4
; bad SQL grammar []; nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'read
   
    from notification' at line 4, mergedContextConfiguration = [MergedContextConfiguration@4082ba93 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]
[INFO][2018-12-0600:10:38][org.springframework.context.support.GenericApplicationContext]Closing org.springframework.context.support.GenericApplicationContext@5dfcfece: startup date [Thu Dec 06 00:10:36 CST 2018]; root of context hierarchy
[INFO][2018-12-0600:16:06][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [spring/spring-mybatis.xml]
[INFO][2018-12-0600:16:06][org.springframework.context.support.GenericApplicationContext]Refreshing org.springframework.context.support.GenericApplicationContext@5dfcfece: startup date [Thu Dec 06 00:16:06 CST 2018]; root of context hierarchy
[INFO][2018-12-0600:16:06][org.springframework.beans.factory.config.PropertyPlaceholderConfigurer]Loading properties file from class path resource [properties/jdbc.properties]
[INFO][2018-12-0600:16:06][org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor]JSR-330 'javax.inject.Inject' annotation found and supported for autowiring
[INFO][2018-12-0600:16:07][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Began transaction (1) for test context [DefaultTestContext@76494737 testClass = UserTest, testInstance = mapper.UserTest@4a003cbe, testMethod = testnoti@UserTest, testException = [null], mergedContextConfiguration = [MergedContextConfiguration@4082ba93 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]; transaction manager [org.springframework.jdbc.datasource.DataSourceTransactionManager@17fc391b]; rollback [true]
[INFO][2018-12-0600:16:07][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [org/springframework/jdbc/support/sql-error-codes.xml]
[INFO][2018-12-0600:16:07][org.springframework.jdbc.support.SQLErrorCodesFactory]SQLErrorCodes loaded: [DB2, Derby, H2, HSQL, Informix, MS-SQL, MySQL, Oracle, PostgreSQL, Sybase]
[INFO][2018-12-0600:16:07][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Rolled back transaction after test execution for test context [DefaultTestContext@76494737 testClass = UserTest, testInstance = mapper.UserTest@4a003cbe, testMethod = testnoti@UserTest, testException = org.springframework.jdbc.BadSqlGrammarException: 
### Error querying database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'read
   
    from notification' at line 4
### The error may exist in file [C:\Users\chwangteng\Documents\GitHub\Report\target\classes\mybatis\mapper\NotificationMapper.xml]
### The error may involve com.chwangteng.www.mapper.NotificationMapper.selectByExample-Inline
### The error occurred while setting parameters
### SQL: select                 id, user_id, user_type, generate_time, title, content, read         from notification
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'read
   
    from notification' at line 4
; bad SQL grammar []; nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'read
   
    from notification' at line 4, mergedContextConfiguration = [MergedContextConfiguration@4082ba93 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]
[INFO][2018-12-0600:16:07][org.springframework.context.support.GenericApplicationContext]Closing org.springframework.context.support.GenericApplicationContext@5dfcfece: startup date [Thu Dec 06 00:16:06 CST 2018]; root of context hierarchy
[INFO][2018-12-0600:27:10][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [spring/spring-mybatis.xml]
[INFO][2018-12-0600:27:10][org.springframework.context.support.GenericApplicationContext]Refreshing org.springframework.context.support.GenericApplicationContext@5dfcfece: startup date [Thu Dec 06 00:27:10 CST 2018]; root of context hierarchy
[INFO][2018-12-0600:27:10][org.springframework.beans.factory.config.PropertyPlaceholderConfigurer]Loading properties file from class path resource [properties/jdbc.properties]
[INFO][2018-12-0600:27:10][org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor]JSR-330 'javax.inject.Inject' annotation found and supported for autowiring
[INFO][2018-12-0600:27:11][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Began transaction (1) for test context [DefaultTestContext@76494737 testClass = UserTest, testInstance = mapper.UserTest@4a003cbe, testMethod = testnoti@UserTest, testException = [null], mergedContextConfiguration = [MergedContextConfiguration@4082ba93 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]; transaction manager [org.springframework.jdbc.datasource.DataSourceTransactionManager@17fc391b]; rollback [true]
[INFO][2018-12-0600:27:11][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [org/springframework/jdbc/support/sql-error-codes.xml]
[INFO][2018-12-0600:27:11][org.springframework.jdbc.support.SQLErrorCodesFactory]SQLErrorCodes loaded: [DB2, Derby, H2, HSQL, Informix, MS-SQL, MySQL, Oracle, PostgreSQL, Sybase]
[INFO][2018-12-0600:27:11][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Rolled back transaction after test execution for test context [DefaultTestContext@76494737 testClass = UserTest, testInstance = mapper.UserTest@4a003cbe, testMethod = testnoti@UserTest, testException = org.springframework.jdbc.BadSqlGrammarException: 
### Error querying database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'read
   
    from motification' at line 4
### The error may exist in file [C:\Users\chwangteng\Documents\GitHub\Report\target\classes\mybatis\mapper\MotificationMapper.xml]
### The error may involve com.chwangteng.www.mapper.MotificationMapper.selectByExample-Inline
### The error occurred while setting parameters
### SQL: select                 id, user_id, user_type, generate_time, title, content, read         from motification
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'read
   
    from motification' at line 4
; bad SQL grammar []; nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'read
   
    from motification' at line 4, mergedContextConfiguration = [MergedContextConfiguration@4082ba93 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]
[INFO][2018-12-0600:27:11][org.springframework.context.support.GenericApplicationContext]Closing org.springframework.context.support.GenericApplicationContext@5dfcfece: startup date [Thu Dec 06 00:27:10 CST 2018]; root of context hierarchy
[INFO][2018-12-0600:30:03][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [spring/spring-mybatis.xml]
[INFO][2018-12-0600:30:04][org.springframework.context.support.GenericApplicationContext]Refreshing org.springframework.context.support.GenericApplicationContext@5dfcfece: startup date [Thu Dec 06 00:30:04 CST 2018]; root of context hierarchy
[INFO][2018-12-0600:30:04][org.springframework.beans.factory.config.PropertyPlaceholderConfigurer]Loading properties file from class path resource [properties/jdbc.properties]
[INFO][2018-12-0600:30:04][org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor]JSR-330 'javax.inject.Inject' annotation found and supported for autowiring
[INFO][2018-12-0600:30:05][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Began transaction (1) for test context [DefaultTestContext@76494737 testClass = UserTest, testInstance = mapper.UserTest@4a003cbe, testMethod = testnoti@UserTest, testException = [null], mergedContextConfiguration = [MergedContextConfiguration@4082ba93 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]; transaction manager [org.springframework.jdbc.datasource.DataSourceTransactionManager@17fc391b]; rollback [true]
[INFO][2018-12-0600:30:05][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [org/springframework/jdbc/support/sql-error-codes.xml]
[INFO][2018-12-0600:30:05][org.springframework.jdbc.support.SQLErrorCodesFactory]SQLErrorCodes loaded: [DB2, Derby, H2, HSQL, Informix, MS-SQL, MySQL, Oracle, PostgreSQL, Sybase]
[INFO][2018-12-0600:30:05][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Rolled back transaction after test execution for test context [DefaultTestContext@76494737 testClass = UserTest, testInstance = mapper.UserTest@4a003cbe, testMethod = testnoti@UserTest, testException = org.springframework.jdbc.BadSqlGrammarException: 
### Error querying database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'read
   
    from motification' at line 4
### The error may exist in file [C:\Users\chwangteng\Documents\GitHub\Report\target\classes\mybatis\mapper\MotificationMapper.xml]
### The error may involve com.chwangteng.www.mapper.MotificationMapper.selectByExample-Inline
### The error occurred while setting parameters
### SQL: select                 id, user_id, user_type, generate_time, title, content, read         from motification
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'read
   
    from motification' at line 4
; bad SQL grammar []; nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'read
   
    from motification' at line 4, mergedContextConfiguration = [MergedContextConfiguration@4082ba93 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]
[INFO][2018-12-0600:30:05][org.springframework.context.support.GenericApplicationContext]Closing org.springframework.context.support.GenericApplicationContext@5dfcfece: startup date [Thu Dec 06 00:30:04 CST 2018]; root of context hierarchy
[INFO][2018-12-0600:35:09][org.springframework.beans.factory.xml.XmlBeanDefinitionReader]Loading XML bean definitions from class path resource [spring/spring-mybatis.xml]
[INFO][2018-12-0600:35:10][org.springframework.context.support.GenericApplicationContext]Refreshing org.springframework.context.support.GenericApplicationContext@5dfcfece: startup date [Thu Dec 06 00:35:10 CST 2018]; root of context hierarchy
[INFO][2018-12-0600:35:10][org.springframework.beans.factory.config.PropertyPlaceholderConfigurer]Loading properties file from class path resource [properties/jdbc.properties]
[INFO][2018-12-0600:35:10][org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor]JSR-330 'javax.inject.Inject' annotation found and supported for autowiring
[INFO][2018-12-0600:35:11][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Began transaction (1) for test context [DefaultTestContext@76494737 testClass = UserTest, testInstance = mapper.UserTest@4a003cbe, testMethod = testnoti@UserTest, testException = [null], mergedContextConfiguration = [MergedContextConfiguration@4082ba93 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]; transaction manager [org.springframework.jdbc.datasource.DataSourceTransactionManager@17fc391b]; rollback [true]
[INFO][2018-12-0600:35:11][org.springframework.test.context.transaction.TransactionalTestExecutionListener]Rolled back transaction after test execution for test context [DefaultTestContext@76494737 testClass = UserTest, testInstance = mapper.UserTest@4a003cbe, testMethod = testnoti@UserTest, testException = [null], mergedContextConfiguration = [MergedContextConfiguration@4082ba93 testClass = UserTest, locations = '{classpath:spring/spring-mybatis.xml}', classes = '{}', contextInitializerClasses = '[]', activeProfiles = '{}', contextLoader = 'org.springframework.test.context.support.DelegatingSmartContextLoader', parent = [null]]]
[INFO][2018-12-0600:35:11][org.springframework.context.support.GenericApplicationContext]Closing org.springframework.context.support.GenericApplicationContext@5dfcfece: startup date [Thu Dec 06 00:35:10 CST 2018]; root of context hierarchy
